\documentclass[12pt]{article}
% \documentstyle{iascars2017}

% \usepackage{iascars2017}

\pagestyle{myheadings} 
\pagenumbering{arabic}
\topmargin 0pt \headheight 23pt \headsep 24.66pt
%\topmargin 11pt \headheight 12pt \headsep 13.66pt
\parindent = 3mm 


\begin{document}


\begin{flushleft}


{\LARGE\bf Statistical challenges for proteogenomic data analysis}


\vspace{1.0cm}

Yusuke Matsui

\begin{description}

\item Division of Systems Biology, Nagoya University Graduate School of Medicine, Japan

\end{description}

\end{flushleft}

%  ***** ADD ENOUGH VERTICAL SPACE HERE TO ENSURE THAT THE *****
%  ***** ABSTRACT (OR MAIN TEXT) STARTS 5 CM BELOW THE TOP *****

\vspace{0.75cm}

\noindent {\bf Abstract}. Large-scale integrated cancer genome and proteome characterization efforts including the cancer genome atlas and clinical proteomic tumor analysis consortium have opened unprecedented opportunities to reveal the comprehensive understandings of cancer biology. An important challenge is organizing our knowledge how the genomic events drive the proteome and phosphoproteome to form phenotypic characteristics. 

However, connecting the genome and the proteome is not straightforward since measuring technologies for the genome and the proteome are quite different and thus the coverage that can be measured is different. Besides, the proteome data usually include massive amount of missing values. These issues include several statistical problems - massive missing data imputation, statistical models to connect the distinct datasets from different measurement technologies, and interpretable statistical models that can explain clinical outcomes.

Currently, we are developing proteogenomic data analysis approaches including the systematic workflow (pipeline) for the cancer protegenomic clinical data. We will present the progression of our current work as well as discussing the statistical approaches, using actual ongoing clinical dataset. 

\vskip 2mm

\noindent {\bf Keywords}.
Bioinformatics, Proteogenomics, Cancer, Multi-omics data, Data analysis


\subsection*{References}

\begin{description}

\item
Philipp, M., {\it et al.} (2016) Proteogenomics connects somatic mutations to signalling in breast cancer. \textit{Nature}, 534:55--62.
\item
Hui, Z., {\it et al.} (2016) Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer. \textit{Cell}, 166:755--765.
\end{description}


\end{document}





