\documentclass[12pt]{article}
% \documentstyle{iascars2017}

% \usepackage{iascars2017}

\pagestyle{myheadings}
\pagenumbering{arabic}
\topmargin 0pt \headheight 23pt \headsep 24.66pt
%\topmargin 11pt \headheight 12pt \headsep 13.66pt
\parindent = 3mm


\begin{document}


\begin{flushleft}


{\LARGE\bf Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data}


\vspace{1.0cm}

Hokeun Sun$^1$

\begin{description}

\item $^1 \;$ Department of Statistics, Pusan National University, 
Busan 46241, Korea


\end{description}

\end{flushleft}

%  ***** ADD ENOUGH VERTICAL SPACE HERE TO ENSURE THAT THE *****
%  ***** ABSTRACT (OR MAIN TEXT) STARTS 5 CM BELOW THE TOP *****

\vspace{0.75cm}

\noindent {\bf Abstract}. In human genetic association studies with high-dimensional microarray data, it has been well known that statistical methods utilizing prior biological network knowledge such as genetic pathways and signaling pathways can outperform other methods that ignore genetic network structures. In recent epigenetic research on case-control association studies, relatively many statistical methods have been proposed to identify cancer-related CpG sites and the corresponding genes from high-dimensional DNA methylation data. However, most of existing methods are not able to utilize genetic networks although methylation levels among linked genes in the networks tend to be highly correlated with each other. In this article, we propose new approach that combines independent component analysis with network-based regularization to identify outcome-related genes for analysis of high-dimensional DNA methylation data. The proposed approach first captures gene-level signals from multiple CpG sites using independent component analysis and then regularizes them to perform gene selection according to given biological network information. In simulation studies, we demonstrated that the proposed approach overwhelms other statistical methods that do not utilize genetic network information in terms of true positive selection. We also applied it to the 450K DNA methylation array data of the four breast invasive carcinoma cancer subtypes from The Cancer Genome Atlas (TCGA) project.

\vskip 2mm

\noindent {\bf Keywords}.
Independent component analysis, network-based regularization, genetic network, DNA methylation, high-dimensional data


%\section{ First-level heading}
%The C98 head 1 style leaves a half-line spacing below a
%first-level heading. There should be one blank line above
%a first-level heading.
%
%\subsection { Second-level heading}
%There should also be one blank line above a second- or
%third-level heading (but no extra space below them).
%
%Do not intent the first paragraph following a heading.
%Second and subsequent paragraphs are indented by one Tab
%character (= 3 mm). If footnotes are used, they should be
%placed at the foot of the page\footnote{ Footnotes are separated
%from the text by a blank line and a printed line of length 3.5 cm.
%They should be printed in 9-point Times Roman in single line spacing.}.
%
%\subsubsection { Third-level heading}
%Please specify references using the conventions
%illustrated below. Each should begin on a new line, and
%second and subsequent lines should be on the same page
%indented by 3 mm.

%\subsection*{References}

%\begin{description}

%\item
%Barnett, J.A., Payne, R.W. and Yarrow, D. (1990).
%\textit{Yeasts: Characteristics and identification: Second Edition.}
%Cambridge: Cambridge University Press.

%\item
%(ed.) Barnett, V., Payne, R. and Steiner, R. (1995).
%\textit{Agricultural Sustainability: Economic, Environmental and
%Statistical Considerations}. Chichester: Wiley.

%\item
%Payne, R.W. (1997).
%\textit{Algorithm AS314 Inversion of matrices Statistics},
%\textbf{46}, 295--298.

%\item
%Payne, R.W. and Welham, S.J. (1990).
%A comparison of algorithms for combination of information in generally
%balanced designs.
%In: \textit{COMPSTAT90 Proceedings in Computational Statistics}, 297--302.
%Heidelberg: Physica-Verlag.

%\end{description}

\end{document}





