<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017</title>
  <meta name="description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="pictures/logo.png" />
  <meta property="og:description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017" />
  <meta name="github-repo" content="jmcurran/NZSA-IASC-Prog" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017" />
  
  <meta name="twitter:description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017" />
  <meta name="twitter:image" content="pictures/logo.png" />

<meta name="author" content="Auckland New Zealand">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="programme-and-abstracts-for-wednesday-13th-of-december.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">New Zealand Statistical Association and the International Association of Statistical Computing (Asian Region) Joint Conference 2017</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html"><i class="fa fa-check"></i>Programme At A Glance</a><ul>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#monday"><i class="fa fa-check"></i>Monday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#tuesday"><i class="fa fa-check"></i>Tuesday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#wednesday"><i class="fa fa-check"></i>Wednesday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#thursday"><i class="fa fa-check"></i>Thursday</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-monday-11th-of-december.html#programme-and-abstracts-for-monday-11th-of-december">Programme And Abstracts For Monday 11<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html"><i class="fa fa-check"></i>Effect of area level deprivation on body mass index: analysis of NZ Health Surveys</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#calendar-based-graphics-for-visualising-peoples-daily-schedules"><i class="fa fa-check"></i>Calendar-based graphics for visualising people’s daily schedules</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#nonparametric-test-for-volatility-in-clustered-multiple-time-series-abstract"><i class="fa fa-check"></i>Nonparametric Test for Volatility in Clustered Multiple Time Series Abstract</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#igess-a-statistical-approach-to-integrating-individual-level-genotype-data-and-summary-statistics-in-genome-wide-association-studies"><i class="fa fa-check"></i>IGESS: A Statistical Approach to Integrating Individual Level Genotype Data and Summary Statistics in Genome Wide Association Studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#clustering-using-nonparametric-mixtures-and-mode-identification"><i class="fa fa-check"></i>Clustering using Nonparametric Mixtures and Mode Identification</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#bayesian-curve-fitting-for-discontinuous-function-using-overcomplete-representation-with-multiple-kernels"><i class="fa fa-check"></i>Bayesian curve fitting for discontinuous function using overcomplete representation with multiple kernels</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimation-of-a-semiparametric-spatiotemporal-models-with-mixed-frequency"><i class="fa fa-check"></i>Estimation of a Semiparametric Spatiotemporal Models with Mixed Frequency</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#lsmm-a-statistical-approach-to-integrating-functional-annotations-with-genome-wide-association-studies"><i class="fa fa-check"></i>LSMM: A statistical approach to integrating functional annotations with genome-wide association studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimation-of-a-high-dimensional-covariance-matrix"><i class="fa fa-check"></i>Estimation of a High-Dimensional Covariance Matrix</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#innovative-bayesian-estimation-in-the-von-mises-distribution"><i class="fa fa-check"></i>Innovative Bayesian Estimation in the von-Mises Distribution</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#evidence-of-climate-change-from-nonparametric-change-point-analysis"><i class="fa fa-check"></i>Evidence of Climate Change from Nonparametric Change-point Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#joint-analysis-of-individual-level-genotype-data-and-summary-statistics-by-leveraging-pleiotropy"><i class="fa fa-check"></i>Joint Analysis of Individual Level Genotype Data and Summary Statistics by Leveraging Pleiotropy</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#genetic-map-estimation-using-hidden-markov-models-in-the-presence-of-partially-observed-information"><i class="fa fa-check"></i>Genetic map estimation using hidden Markov models in the presence of partially observed information</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#a-simple-method-for-grouping-patients-based-on-historical-doses"><i class="fa fa-check"></i>A simple method for grouping patients based on historical doses</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#semiparametric-mixed-analysis-of-covariance-model"><i class="fa fa-check"></i>Semiparametric Mixed Analysis of Covariance Model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#adaptive-false-discovery-rate-regression-with-application-in-integrative-analysis-of-large-scale-genomic-data"><i class="fa fa-check"></i>Adaptive False Discovery Rate regression with application in integrative analysis of large-scale genomic data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#vector-generalized-linear-time-series-models"><i class="fa fa-check"></i>Vector Generalized Linear Time Series Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#local-canonical-correlation-analysis-for-multimodal-labeled-data"><i class="fa fa-check"></i>Local Canonical Correlation Analysis for Multimodal Labeled Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#canonical-covariance-analysis-for-mixed-numerical-and-categorical-three-way-three-mode-data"><i class="fa fa-check"></i>Canonical Covariance Analysis for Mixed Numerical and Categorical Three-way Three-mode Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#variable-selection-algorithms"><i class="fa fa-check"></i>Variable Selection Algorithms</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimating-causal-structures-for-continuous-and-discrete-variables"><i class="fa fa-check"></i>Estimating causal structures for continuous and discrete variables</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#incorporating-genetic-networks-into-case-control-association-studies-with-high-dimensional-dna-methylation-data"><i class="fa fa-check"></i>Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#adaptive-model-checking-for-functional-single-index-models"><i class="fa fa-check"></i>Adaptive Model Checking for Functional Single-Index Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#on-optimal-group-testing-designs-prevalence-estimation-cost-considerations-and-dilution-effects"><i class="fa fa-check"></i>On Optimal Group Testing Designs: Prevalence Estimation, Cost Considerations, and Dilution Effects</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#the-use-of-bayesian-networks-in-grape-yield-prediction"><i class="fa fa-check"></i>The Use of Bayesian Networks in Grape Yield Prediction</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="abstract.html"><a href="abstract.html"><i class="fa fa-check"></i><b>1</b> Abstract</a><ul>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#pattern-prediction-for-time-series-data-with-change-points"><i class="fa fa-check"></i>Pattern Prediction for Time Series Data with Change Points</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#test-for-genomic-imprinting-effects-on-the-x-chromosome"><i class="fa fa-check"></i>Test for Genomic Imprinting Effects on the X Chromosome</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#fluctuation-reduction-of-value-at-risk-estimation-and-its-applications"><i class="fa fa-check"></i>Fluctuation Reduction of Value-at-Risk Estimation and its Applications</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#meta-analytic-principal-component-analysis-in-integrative-omics-application"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#estimation-of-animal-density-from-acoustic-detections"><i class="fa fa-check"></i>Estimation of animal density from acoustic detections</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#mixed-models-for-complex-survey-data"><i class="fa fa-check"></i>Mixed models for complex survey data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#regression-with-random-effects-for-analysing-correlated-survival-data-application-to-disease-recurrences"><i class="fa fa-check"></i>Regression with random effects for analysing correlated survival data: application to disease recurrences</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#genetic-predictors-underlying-long-term-cognitive-recovery-following-mild-traumatic-brain-injury"><i class="fa fa-check"></i>Genetic predictors underlying long-term cognitive recovery following mild traumatic brain injury</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#bayesian-structure-selection-for-vector-autoregression-model"><i class="fa fa-check"></i>Bayesian Structure Selection for Vector Autoregression Model</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#phylogenetic-tree-based-microbiome-association-test"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#talk-data-to-me"><i class="fa fa-check"></i>Talk Data To Me</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#smooth-nonparametric-regression-under-shape-restrictions"><i class="fa fa-check"></i>Smooth Nonparametric Regression under Shape Restrictions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#elastic-band-transform-a-new-approach-to-multiscale-visualization"><i class="fa fa-check"></i>Elastic-Band Transform: A New Approach to Multiscale Visualization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#a-computational-tool-for-detecting-copy-number-variations-from-whole-genome-and-targeted-exome-sequencing"><i class="fa fa-check"></i>A Computational Tool for Detecting Copy Number Variations from Whole Genome and Targeted Exome Sequencing</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area"><i class="fa fa-check"></i>Flight to relative safety: Learning from a no-arbitrage network of yield curves model of the euro area</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#robustness-of-temperature-reconstruction-for-past-500-years"><i class="fa fa-check"></i>Robustness of Temperature Reconstruction for Past 500 Years</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#nonparametric-causal-inference-by-the-kernel-method"><i class="fa fa-check"></i>Nonparametric causal inference by the kernel method</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#a-unified-regularized-group-pls-algorithm-scalable-to-big-data"><i class="fa fa-check"></i>A Unified Regularized Group PLS Algorithm Scalable to Big Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#meta-analysis-with-symbolic-data-analysis-and-its-application-for-clinical-data"><i class="fa fa-check"></i>Meta-analysis with symbolic data analysis and its application for clinical data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#scoring-rules-for-prediction-and-classification-challenges"><i class="fa fa-check"></i>Scoring rules for prediction and classification challenges</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#real-time-transit-network-modelling-for-improved-arrival-time-predictions"><i class="fa fa-check"></i>Real-time transit network modelling for improved arrival time predictions</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#sparse-group-subgroup-partial-least-squares-with-application-to-genomic-data"><i class="fa fa-check"></i>Sparse Group-subgroup Partial Least Squares with Application to Genomic Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#big-sir-a-sliced-inverse-regression-approach-for-massive-data"><i class="fa fa-check"></i>BIG-SIR a Sliced Inverse Regression Approach for Massive Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#comparison-of-tests-of-mean-difference-in-longitudinal-data-based-on-block-resampling-methods"><i class="fa fa-check"></i>Comparison of Tests of Mean Difference in Longitudinal Data Based on Block Resampling Methods</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#modeling-of-document-abstraction-using-association-rule-based-characterization"><i class="fa fa-check"></i>Modeling of Document Abstraction Using Association Rule Based Characterization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#bayesian-survival-analysis-of-batsmen-in-test-cricket"><i class="fa fa-check"></i>Bayesian survival analysis of batsmen in Test cricket</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#covariate-discretisation-on-big-data"><i class="fa fa-check"></i>Covariate Discretisation on Big Data</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#visualization-and-statistical-modeling-of-financial-big-data"><i class="fa fa-check"></i>Visualization and Statistical Modeling of Financial Big Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#analysis-of-spatial-data-with-a-gaussian-mixture-markov-random-field-model"><i class="fa fa-check"></i>Analysis of Spatial Data with a Gaussian Mixture Markov Random Field Model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#my-knee-still-hurts-the-statistical-pathway-to-the-development-of-a-clinical-decision-aid"><i class="fa fa-check"></i>My knee still hurts; the statistical pathway to the development of a clinical decision aid</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#programme-and-abstracts-for-tuesday-12th-of-december">Programme And Abstracts For Tuesday 12<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html"><i class="fa fa-check"></i>R&amp;D policy regimes in France: New evidence from a spatio-temporal analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysing-scientific-collaborations-of-new-zealand-institutions-using-scopus-bibliometric-data"><i class="fa fa-check"></i>Analysing Scientific Collaborations of New Zealand Institutions using Scopus Bibliometric Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#a-practitioners-guide-to-deep-learning-for-predictive-analytics-on-structured-data"><i class="fa fa-check"></i>A Practitioners Guide to Deep Learning for Predictive Analytics on Structured Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysis-of-multivariate-binary-longitudinal-data-metabolic-syndrome-during-menopausal-transition"><i class="fa fa-check"></i>Analysis of Multivariate Binary Longitudinal Data: Metabolic Syndrome During Menopausal Transition</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#clustering-of-curves-on-a-spatial-domain-using-a-bayesian-partitioning-model"><i class="fa fa-check"></i>Clustering of curves on a spatial domain using a Bayesian partitioning model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#the-uncomfortable-entrepreneurs-bad-working-conditions-and-entrepreneurial-commitment"><i class="fa fa-check"></i>The Uncomfortable Entrepreneurs: Bad Working Conditions and Entrepreneurial Commitment</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#spatial-surveillance-with-scan-statistics-by-controlling-the-false-discovery-rate"><i class="fa fa-check"></i>Spatial Surveillance with Scan Statistics by Controlling the False Discovery Rate</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#statistical-models-for-the-source-attribution-of-zoonotic-diseases-a-study-of-campylobacteriosis"><i class="fa fa-check"></i>Statistical models for the source attribution of zoonotic diseases: A study of campylobacteriosis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#family-structure-and-academic-achievements-of-high-school-students-in-tonga"><i class="fa fa-check"></i>Family structure and academic achievements of high school students in Tonga</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#identifying-clusters-of-patients-with-diabetes-using-a-markov-birth-death-process"><i class="fa fa-check"></i>Identifying Clusters Of Patients With Diabetes Using a Markov Birth-Death Process</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-temporal-density-estimation-using-autoregressive-species-sampling-models"><i class="fa fa-check"></i>Bayesian Temporal Density Estimation using Autoregressive Species Sampling Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#intensity-estimation-of-spatial-point-processes-based-on-area-aggregated-data"><i class="fa fa-check"></i>Intensity Estimation of Spatial Point Processes Based on Area-Aggregated Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-inference-for-population-attributable-measures"><i class="fa fa-check"></i>Bayesian Inference for Population Attributable Measures</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#latent-variable-models-and-multivariate-binomial-data"><i class="fa fa-check"></i>Latent variable models and multivariate binomial data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysis-of-a-brief-telephone-intervention-for-problem-gambling-and-examining-the-impact-on-co-existing-depression"><i class="fa fa-check"></i>Analysis of a brief telephone intervention for problem gambling and examining the impact on co-existing depression?</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#prior-based-bayesian-information-criterion"><i class="fa fa-check"></i>Prior-based Bayesian Information Criterion</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#correlated-defaults-with-a-distance-to-default"><i class="fa fa-check"></i>Correlated Defaults with a Distance to Default</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#geographically-weighted-principal-component-analysis-for-spatio-temporal-statistical-dataset"><i class="fa fa-check"></i>Geographically Weighted Principal Component Analysis for Spatio-temporal Statistical Dataset</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#dimensionality-reduction-of-multivariate-data-for-bayesian-analysis"><i class="fa fa-check"></i>Dimensionality Reduction of Multivariate Data for Bayesian Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#an-information-criterion-for-prediction-with-auxiliary-variables-under-covariate-shift"><i class="fa fa-check"></i>An information criterion for prediction with auxiliary variables under covariate shift</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#adjusting-for-linkage-bias-in-the-analysis-of-record-linked-data"><i class="fa fa-check"></i>Adjusting for Linkage Bias in the Analysis of Record-Linked Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-semiparametric-hierarchical-models-for-longitudinal-data-analysis-with-application-to-dose-response-studies"><i class="fa fa-check"></i>Bayesian semiparametric hierarchical models for longitudinal data analysis with application to dose-response studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#optimizing-junior-rugby-weight-limits"><i class="fa fa-check"></i>Optimizing Junior Rugby Weight Limits</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#spatial-scan-statistics-for-matched-case-control-data"><i class="fa fa-check"></i>Spatial scan statistics for matched case-control data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#whitebait-in-all-its-varieties-one-fish-two-fish-three-four-five-fish."><i class="fa fa-check"></i>Whitebait in all its varieties: one fish, two fish, three, four, five fish.</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#an-advanced-approach-for-time-series-forecasting-using-deep-learning"><i class="fa fa-check"></i>An Advanced Approach for Time Series Forecasting using Deep Learning</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#asking-about-sex-in-general-health-surveys-comparing-the-methods-and-findings-of-the-2010-health-survey-for-england-with-those-of-the-third-national-survey-of-sexual-attitudes-and-lifestyles"><i class="fa fa-check"></i>Asking about Sex in General Health Surveys: Comparing the Methods and Findings of the 2010 Health Survey for England with Those of the Third National Survey of Sexual Attitudes and Lifestyles</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#tolerance-limits-for-the-reliability-of-semiconductor-devices-using-longitudinal-data"><i class="fa fa-check"></i>Tolerance limits for the reliability of semiconductor devices using longitudinal data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#model-checking-for-regressions-a-local-smoothing-based-global-smoothing-test"><i class="fa fa-check"></i>Model-checking for regressions: a local smoothing-based global smoothing test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#breeding-value-estimation-in-partially-genotyped-populations"><i class="fa fa-check"></i>Breeding Value Estimation in Partially-Genotyped Populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bivas-a-scalable-bayesian-method-for-bi-level-variable-selection"><i class="fa fa-check"></i>BIVAS: A scalable Bayesian method for Bi-level Variable Selection</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#ranking-potential-shoplifters-in-real-time"><i class="fa fa-check"></i>Ranking Potential Shoplifters in Real Time</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#two-stage-approach-to-data-driven-subgroup-identification-in-clinical-trials"><i class="fa fa-check"></i>Two stage approach to data-driven subgroup identification in clinical trials</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#estimation-and-prediction-for-a-distribution-with-bathtub-shape-under-progressive-first-failure-censoring"><i class="fa fa-check"></i>Estimation and prediction for a distribution with bathtub shape under progressive first failure censoring</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#inverse-regression-for-multivariate-functional-data"><i class="fa fa-check"></i>Inverse Regression for Multivariate Functional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#including-covariate-estimation-error-when-predicting-species-distributions-a-simulation-exercise-using-template-model-builder"><i class="fa fa-check"></i>Including covariate estimation error when predicting species distributions: a simulation exercise using Template Model Builder</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#adjusted-adaptive-index-model-for-binary-response"><i class="fa fa-check"></i>Adjusted adaptive index model for binary response</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#factors-influencing-on-growth-of-garments-industry-in-bangladesh"><i class="fa fa-check"></i>Factors Influencing on Growth of Garments Industry in Bangladesh</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#comparison-of-exact-and-approximate-testing-procedures-in-clinical-trials-with-multiple-binary-endpoints"><i class="fa fa-check"></i>Comparison of Exact and Approximate Testing Procedures in Clinical Trials with Multiple Binary Endpoints</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-optimum-warranty-length-under-type-ii-unified-hybrid-censoring-scheme"><i class="fa fa-check"></i>Bayesian optimum warranty length under Type-II unified hybrid censoring scheme</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#multiple-function-on-function-linear-regression-with-application-to-weather-forecast-calibration"><i class="fa fa-check"></i>Multiple function-on-function linear regression with application to weather forecast calibration</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#modelling-the-distribution-of-lifetime-using-compound-time-homogenous-poisson-process"><i class="fa fa-check"></i>Modelling the distribution of lifetime using compound time-homogenous Poisson process</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#detecting-change-points-in-the-stress-strength-reliability-pxy"><i class="fa fa-check"></i>Detecting Change-Points in the Stress-Strength Reliability P(X&lt;Y)</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#new-zealand-crime-and-victims-survey-filling-the-knowledge-gap"><i class="fa fa-check"></i>New Zealand Crime and Victims Survey: filling the knowledge gap</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#missing-data-in-randomised-control-trials-stepped-multiple-imputation"><i class="fa fa-check"></i>Missing Data in Randomised Control Trials: Stepped Multiple Imputation</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#programme-and-abstracts-for-wednesday-13th-of-december">Programme And Abstracts For Wednesday 13<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html"><i class="fa fa-check"></i>Promoting your R package</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-smoothing-filter-modelling-approach-for-time-series"><i class="fa fa-check"></i>A smoothing filter modelling approach for time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#online-learning-for-bayesian-nonparametrics-weakly-conjugate-approximation"><i class="fa fa-check"></i>Online Learning for Bayesian Nonparametrics: Weakly Conjugate Approximation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#improving-the-production-cycle-at-stats-nz-with-rstudio"><i class="fa fa-check"></i>Improving the production cycle at Stats NZ with RStudio</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#sparse-common-component-analysis"><i class="fa fa-check"></i>Sparse Common Component Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#random-search-global-optimization-using-random-forests"><i class="fa fa-check"></i>Random Search Global Optimization using Random Forests</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#gridsvg-then-and-now"><i class="fa fa-check"></i>gridSVG: Then and Now</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#probabilistic-outlier-detection-and-visualization-of-smart-meter-data"><i class="fa fa-check"></i>Probabilistic outlier detection and visualization of smart meter data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#the-joint-models-for-nonlinear-longitudinal-and-time-to-event-data-using-penalized-splines-a-bayesian-approach"><i class="fa fa-check"></i>The joint models for nonlinear longitudinal and time-to-event data using penalized splines: A Bayesian approach</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-a-powerful-analysis-tool-to-improve-official-statistics-in-romania"><i class="fa fa-check"></i>R – A powerful analysis tool to improve Official Statistics in Romania</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#robust-principal-expectile-component-analysis"><i class="fa fa-check"></i>Robust Principal Expectile Component Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bringing-multimix-from-fortran-to-r"><i class="fa fa-check"></i>Bringing Multimix from Fortran to R</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#specification-of-garch-model-under-asymmetric-error-innovations"><i class="fa fa-check"></i>SPECIFICATION OF GARCH MODEL UNDER ASYMMETRIC ERROR INNOVATIONS</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#the-joint-models-for-nonlinear-longitudinal-and-time-to-event-data-using-penalized-splines-a-bayesian-approach-1"><i class="fa fa-check"></i>The joint models for nonlinear longitudinal and time-to-event data using penalized splines: A Bayesian approach</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#statistical-disclosure-control-with-r-traditional-methods-and-synthetic-data"><i class="fa fa-check"></i>Statistical Disclosure Control with R: Traditional Methods and Synthetic Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-max-type-multivariate-two-sample-baumgartner-statistic"><i class="fa fa-check"></i>A Max-Type Multivariate Two-Sample Baumgartner Statistic</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#deep-learning-high-dimensional-covariance-matrices"><i class="fa fa-check"></i>Deep Learning High-Dimensional Covariance Matrices</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-in-industry-application-on-pipe-renewal-planning"><i class="fa fa-check"></i>R in Industry – Application on Pipe Renewal Planning</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#empirical-comparison-of-some-algorithms-for-automatic-univariate-arma-modeling-using-rcmdrplugin.spss"><i class="fa fa-check"></i>Empirical comparison of some algorithms for automatic univariate ARMA modeling using RcmdrPlugin.SPSS</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-analyses-of-non-homogeneous-gaussian-hidden-markov-models"><i class="fa fa-check"></i>Bayesian Analyses of Non-homogeneous Gaussian Hidden Markov Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#imputation-of-the-2016-economic-census-for-business-activity-in-japan"><i class="fa fa-check"></i>Imputation of the 2016 Economic Census for Business Activity in Japan</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#simultaneous-test-for-mean-vectors-and-covariance-matrices-in-high-dimensional-settings"><i class="fa fa-check"></i>Simultaneous test for mean vectors and covariance matrices in high-dimensional settings</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#applying-active-learning-procedure-to-drug-consumption-data"><i class="fa fa-check"></i>Applying Active Learning Procedure to Drug Consumption Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-for-everything"><i class="fa fa-check"></i>R for Everything</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-package-for-new-two-stage-methods-in-forecasting-time-series-with-multiple-seasonality"><i class="fa fa-check"></i>R Package for New Two-Stage Methods in Forecasting Time Series with Multiple Seasonality</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#performance-of-bayesian-credible-interval-for-binomial-proportion-using-logit-transformation"><i class="fa fa-check"></i>Performance of Bayesian Credible Interval for Binomial Proportion using Logit Transformation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#analysis-of-official-microdata-using-secure-statistical-computation-system"><i class="fa fa-check"></i>Analysis of Official Microdata Using Secure Statistical Computation System</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#high-dimensional-asymptotics-for-the-naive-canonical-correlation-coefficient"><i class="fa fa-check"></i>High Dimensional Asymptotics for the Naive Canonical Correlation Coefficient</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#presenting-flexi-a-statistical-program-for-fitting-variance-models"><i class="fa fa-check"></i>Presenting Flexi, A Statistical Program for Fitting Variance Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#cluster-wise-regression-models-combined-by-a-quasi-linear-function"><i class="fa fa-check"></i>Cluster-wise regression models combined by a quasi-linear function</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#hierarchical-structural-component-analysis-of-gene-environment-interactions"><i class="fa fa-check"></i>Hierarchical Structural Component Analysis of Gene-Environment Interactions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series-1"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#cross-covariance-estimation-for-integration-of-multi-omics-data"><i class="fa fa-check"></i>Cross covariance estimation for integration of multi-omics data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#relationships-between-linguistic-characteristics-and-the-use-of-maori-loanwords-in-new-zealand-english."><i class="fa fa-check"></i>Relationships between linguistic characteristics and the use of Māori loanwords in New Zealand English.</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#transfer-regression-and-predictive-distributions"><i class="fa fa-check"></i>Transfer Regression and Predictive Distributions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#fluctuation-reduction-of-value-at-risk-estimation-and-its-applications-1"><i class="fa fa-check"></i>Fluctuation Reduction of Value-at-Risk Estimation and its Applications</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations-1"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-analysis-for-fitting-zero-inflated-count-data-with-data-augmentation"><i class="fa fa-check"></i>Bayesian analysis for fitting zero-inflated count data with data augmentation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#regularized-noise-reduction-methodology-for-high-dimensional-data"><i class="fa fa-check"></i>Regularized Noise-Reduction Methodology for High-Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#computation-of-influence-functions-for-robust-statistics"><i class="fa fa-check"></i>Computation of influence functions for robust statistics</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#adaptive-model-averaging-in-high-dimensional-linear-regression"><i class="fa fa-check"></i>Adaptive Model Averaging in High-Dimensional Linear Regression</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#meta-analytic-principal-component-analysis-in-integrative-omics-application-1"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#model-based-clustering-for-multivariate-categorical-data-with-dimension-reduction"><i class="fa fa-check"></i>Model-based clustering for multivariate categorical data with dimension reduction</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#selecting-the-number-of-principal-components"><i class="fa fa-check"></i>Selecting the number of principal components</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#rolling-survival-extrapolation-algorithm-for-estimating-life-years-lost-among-subjects-exposed-to-long-term-air-pollution"><i class="fa fa-check"></i>Rolling survival extrapolation algorithm for estimating life years lost among subjects exposed to long-term air pollution</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#enhancing-the-flexibility-of-regression-modeling-by-liquid-association"><i class="fa fa-check"></i>Enhancing the flexibility of regression modeling by liquid association</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area-1"><i class="fa fa-check"></i>Flight to relative safety: Learning from a no-arbitrage network of yield curves model of the euro area</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#phylogenetic-tree-based-microbiome-association-test-1"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#towards-a-sparse-scalable-and-stably-positive-definite-inverse-covariance-estimator"><i class="fa fa-check"></i>Towards a sparse, scalable, and stably positive definite (inverse) covariance estimator</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#high-mortality-predictions-with-lines-or-curves-fitted-to-over-dispersed-exposure-mortality-data"><i class="fa fa-check"></i>High Mortality Predictions with Lines or Curves Fitted to Over-Dispersed Exposure-Mortality Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#lattice-polytope-samplers"><i class="fa fa-check"></i>Lattice Polytope Samplers</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#statistical-modelling-and-analysis-of-cosmic-microwave-background-data"><i class="fa fa-check"></i>Statistical Modelling and Analysis of Cosmic Microwave Background Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#measure-of-departure-from-marginal-average-point-symmetry-for-two-way-contingency-tables-with-ordered-categories"><i class="fa fa-check"></i>Measure of Departure from Marginal Average Point-Symmetry for Two-Way Contingency Tables with Ordered Categories</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#sparse-estimates-from-dense-precision-matrix-posteriors"><i class="fa fa-check"></i>Sparse estimates from dense precision matrix posteriors</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#dimension-reduction-strategies-for-modeling-bi-clustered-high-dimensional-data"><i class="fa fa-check"></i>Dimension Reduction Strategies for Modeling Bi-clustered High Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#computing-entropies-with-nested-sampling"><i class="fa fa-check"></i>Computing Entropies with Nested Sampling</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#spline-based-drift-models-for-high-temperature-operating-life-tests"><i class="fa fa-check"></i>Spline-based Drift Models for High Temperature Operating Life Tests</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-new-approach-to-distribution-free-tests-in-contingency-tables"><i class="fa fa-check"></i>A new approach to distribution free tests in contingency tables</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-continuous-space-time-model-of-burglaries"><i class="fa fa-check"></i>Bayesian continuous space-time model of Burglaries</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#modified-gene-shaving-algorithm---a-dimension-reduction-and-clustering-method"><i class="fa fa-check"></i>Modified Gene Shaving Algorithm - A Dimension REduction and Clustering Method</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-simple-method-to-construct-confidence-bands-in-functional-linear-regression"><i class="fa fa-check"></i>A simple method to construct confidence bands in functional linear regression</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#separation-of-symmetry-for-square-contingency-tables-with-ordinal-categories"><i class="fa fa-check"></i>Separation of symmetry for square contingency tables with ordinal categories</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#an-overview-of-the-correspondence-analysis-family"><i class="fa fa-check"></i>An overview of the correspondence analysis family</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#testing-for-presence-of-clustering-effect-in-multilevel-model-with-high-dimensional-predictors"><i class="fa fa-check"></i>TESTING FOR PRESENCE OF CLUSTERING EFFECT IN MULTILEVEL MODEL WITH HIGH-DIMENSIONAL PREDICTORS</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-thursday-14th-of-december.html#programme-and-abstracts-for-thursday-14th-of-december">Programme And Abstracts For Thursday 14<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html"><i class="fa fa-check"></i>Penalized Vector Generalized Additive Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#author-name-identification-for-evaluating-research-performance-of-institutes"><i class="fa fa-check"></i>Author Name Identification for Evaluating Research Performance of Institutes</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#cross-covariance-estimation-for-integration-of-multi-omics-data-1"><i class="fa fa-check"></i>Cross covariance estimation for integration of multi-omics data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#evaluation-of-spatial-cluster-detection-method-based-on-all-geographical-linkage-patterns"><i class="fa fa-check"></i>Evaluation of spatial cluster detection method based on all geographical linkage patterns</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#bayesian-static-parameter-inference-for-partially-observed-stochastic-systems"><i class="fa fa-check"></i>Bayesian Static Parameter Inference for Partially Observed Stochastic Systems</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations-2"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#consistency-of-linear-mixed-effects-model-selection-with-inconsistent-covariance-parameter-estimators"><i class="fa fa-check"></i>Consistency of Linear Mixed-Effects Model Selection with Inconsistent Covariance Parameter Estimators</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#a-study-of-the-influence-of-articles-in-the-large-scale-citation-network"><i class="fa fa-check"></i>A Study of the Influence of Articles in the Large-Scale Citation Network</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#regularized-noise-reduction-methodology-for-high-dimensional-data-1"><i class="fa fa-check"></i>Regularized Noise-Reduction Methodology for High-Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#genetic-approach-and-statistical-approach-for-association-study-on-dna-data"><i class="fa fa-check"></i>Genetic Approach and Statistical Approach for Association Study on DNA Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#ssrem-a-summary-statistics-based-random-effect-model-to-estimating-heritability-co-heritability-and-effect-sizes-in-gwas-data-analysis"><i class="fa fa-check"></i>SSREM: A Summary-Statistics-based Random Effect Model to Estimating heritability, co-heritability and effect sizes in GWAS data analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#meta-analytic-principal-component-analysis-in-integrative-omics-application-2"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#selecting-generalised-linear-models-under-inequality-constraints"><i class="fa fa-check"></i>Selecting Generalised Linear Models Under Inequality Constraints</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#clustering-of-research-subject-based-on-stochastic-block-model"><i class="fa fa-check"></i>Clustering of research subject based on stochastic block model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#selecting-the-number-of-principal-components-1"><i class="fa fa-check"></i>Selecting the number of principal components</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#symbolic-data-analytical-approach-to-unauthorized-access-logs"><i class="fa fa-check"></i>Symbolic data analytical approach to unauthorized-access logs</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#forward-selection-in-regression-models-based-on-robust-estimation"><i class="fa fa-check"></i>Forward selection in regression models based on robust estimation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#phylogenetic-tree-based-microbiome-association-test-2"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#statistical-generalized-derivative-applied-to-the-profile-likelihood-estimation-in-a-mixture-of-semiparametric-models"><i class="fa fa-check"></i>Statistical generalized derivative applied to the profile likelihood estimation in a mixture of semiparametric models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#structure-of-members-in-the-organization-to-induce-innovation-quantitatively-analyze-the-capability-of-the-organization"><i class="fa fa-check"></i>Structure of Members in the Organization to Induce Innovation: Quantitatively Analyze the Capability of the Organization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#towards-a-sparse-scalable-and-stably-positive-definite-inverse-covariance-estimator-1"><i class="fa fa-check"></i>Towards a sparse, scalable, and stably positive definite (inverse) covariance estimator</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#statistical-challenges-for-proteogenomic-data-analysis"><i class="fa fa-check"></i>Statistical challenges for proteogenomic data analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#feature-selection-in-high-dimensional-models-with-complex-block-structures"><i class="fa fa-check"></i>Feature selection in high-dimensional models with complex block structures</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series-2"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#dissimilarities-between-groups-of-data"><i class="fa fa-check"></i>Dissimilarities between groups of data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#a-package-for-multiple-precision-floating-point-computation-on-r"><i class="fa fa-check"></i>A package for multiple precision floating-point computation on R</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#cluster-wise-regression-models-combined-by-a-quasi-linear-function-1"><i class="fa fa-check"></i>Cluster-wise regression models combined by a quasi-linear function</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#mobile-learning-in-teaching-bioinformatics-for-medical-doctors"><i class="fa fa-check"></i>Mobile Learning in teaching Bioinformatics for medical doctors</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#interactive-visualization-of-aggregated-symbolic-data"><i class="fa fa-check"></i>Interactive visualization of aggregated symbolic data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#an-incomplete-data-fisher-scoring-with-an-acceleration-method"><i class="fa fa-check"></i>An incomplete-data Fisher scoring with an acceleration method</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#clusterwise-low-rank-correlation-analysis-based-on-majorization"><i class="fa fa-check"></i>Clusterwise low-rank correlation analysis based on majorization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#towards-an-informal-test-for-goodness-of-fit"><i class="fa fa-check"></i>Towards an informal test for goodness-of-fit</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#an-ewma-chart-for-monitoring-covariance-matrix-based-on-dissimilarity-index"><i class="fa fa-check"></i>An EWMA chart for monitoring covariance matrix based on dissimilarity index</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#improvement-of-computation-for-nonlinear-multivariate-methods"><i class="fa fa-check"></i>Improvement of Computation for Nonlinear Multivariate Methods</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#three-dimensional-data-visualization-education-with-virtual-reality"><i class="fa fa-check"></i>Three-Dimensional Data Visualization Education with Virtual Reality</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="programme-and-abstracts-for-thursday-14th-of-december" class="section level1 unnumbered">
<h1>Programme And Abstracts For Thursday 14<sup>th</sup> Of December</h1>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:30 098 Lecture Theatre (260-098)
</p>
<div id="penalized-vector-generalized-additive-models" class="section level2 unnumbered">
<h2>Penalized Vector Generalized Additive Models</h2>
<p style="text-align:center">
Thomas Yee<sup>1</sup>, Chanatda Somchit<sup>2</sup>, and Chris Wild<sup>1</sup><br />
</p>
<p><span><strong>Penalized Vector Generalized Additive Models</strong></span></p>
<p>T. W. Yee<span class="math inline">\(^1\)</span> and C. Somchit<span class="math inline">\(^2\)</span> and C. J. Wild<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, University of Auckland, New Zealand</p>
<p><span class="math inline">\(^2 \;\)</span> Phayao University, Thailand</p>
<p><span><strong>Abstract</strong></span>. Over the last two decades generalized additive models (GAMs) have become an indispensible tool for modern data analysis and regression. First-generation GAMs as developed by Hastie and Tibshirani are based on backfitting (e.g., the <code>gam</code> R package). Second-generation GAMs have automatic smoothing parameter selection (e.g., the <code>mgcv</code> package by Simon Wood) and are based on, e.g., P-splines. Until recently, these two implementations were largely confined to the exponential family. However, since the 1990s, the vector generalized linear and additive model (VGLM/VGAM) classes were developed by Yee and coworkers, and these are a much larger class of models. First-generation VGAMs were based on vector splines and vector backfitting. This talk will describe 2nd-generation VGAMs using O-splines and P-splines. We illustrate them by examples, to show that automatic smoothing parameter selection based on optimizing a predictive quantity such as generalized cross validation can be very useful. The speaker’s <code>VGAM</code> package implementation will be described.</p>
<p><span><strong>Keywords</strong></span>. Automatic smoothing parameter selection, O-splines, P-splines, Vector generalized additive models, VGAM R package</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Yee, T.W. (2015). <em>Vector Generalized Linear and Additive Models: With an Implementation in R</em>. New York, USA: Springer.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:30 OGGB4 (260-073)
</p>
</div>
<div id="author-name-identification-for-evaluating-research-performance-of-institutes" class="section level2 unnumbered">
<h2>Author Name Identification for Evaluating Research Performance of Institutes</h2>
<p style="text-align:center">
Tomokazu Fujino<sup>1</sup>, Keisuke Honda<sup>2</sup>, and Hiroka Hamada<sup>2</sup><br />
</p>
<p><span><strong>Author Name Identification for Evaluating Research Performance of Institutes</strong></span></p>
<p>Tomokazu Fujino<span class="math inline">\(^1\)</span>, Keisuke Honda<span class="math inline">\(^2\)</span> and Hiroka Hamada<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Environmental Science, Fukuoka Women’s University, Kasumigaoka, Fukuoka 813-8529, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Institute of Statistical Mathematics, Tachikawa, Tokyo 190-8562, Japan</p>
<p><span><strong>Abstract</strong></span>. In this paper, we propose a new framework to extract a complete list of the articles written by researchers who belong to a specific research or educational institute from an academic document database such as Web of Science and Scopus. In this framework, it is necessary to perform author name identification because the query for the database is based on the author’s name to extract documents written before the author comming to the current institute. The framework is based on the latent dirichlet allocation (LDA), which is a kind of topic modeling, and some techniques and indices such as synonym retrieval and inverse document frequency (IDF) are used for enhancing the framework.</p>
<p><span><strong>Keywords</strong></span>. Institutional Research, Topic Modeling, Latent Dirichlet Allocation</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Tang, L. and Walsh,J.P. (2010). Bibliometric fingerprints: name disambiguation based on approximate structure equivalence of cognitive maps. <em>Scientometrics</em>, 84(3), 763–784.</p>
<p>Strotmann,A., Zhao,D. and Bubela,T. (2009). Author name disambiguation for collaboration network analysis and visualization. <em>Proc. American Society for Information Science and Technology</em>, 46(1), 1–20.</p>
Soler,J.M. (2007). Separating the articles of authors with the same name. <em>Scientometrics</em>, 72(2), 281–290.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:30 OGGB5 (260-051)
</p>
</div>
<div id="cross-covariance-estimation-for-integration-of-multi-omics-data-1" class="section level2 unnumbered">
<h2>Cross covariance estimation for integration of multi-omics data</h2>
<p style="text-align:center">
Johan Lim<sup>1</sup>, Hiromi Koh<sup>2</sup>, and Hyungwon Choi<sup>2</sup><br />
</p>
<p><span><strong>Cross covariance estimation for integration of multi-omics data</strong></span></p>
<p>Johan Lim<span class="math inline">\(^1\)</span>, Hiromi Koh<span class="math inline">\(^2\)</span>, and Hyungwon Choi<span class="math inline">\(^{2}\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, Seoul National University, Seoul, Korea</p>
<p><span class="math inline">\(^2 \;\)</span> School of Public Health, National University of Singapore</p>
<p><span><strong>Abstract</strong></span>. In integrative analysis of multiple types of -omics data, it is often of interest to infer associations between two different types of molecules. The prevailing analysis methods depend on ensemble of brute-force pairwise univariate association tests between the two types, best exemplified by expression quantitative loci (eQTL) mapping. In a nutshell, this problem can be generally framed as a sparse cross-covariance matrix. In this work, we propose a two-stage estimator of cross covariance matrix <span class="math inline">\({\mathbf{\Sigma_{XY}}}\)</span> between <span class="math inline">\(p\)</span>-vector <span class="math inline">\({\mathbf{X}}\)</span> and <span class="math inline">\(q\)</span>-vector <span class="math inline">\({\mathbf{Y}}\)</span>, assuming that the two variables have regulatory relationships and that we know a group structure in the variables in <span class="math inline">\({\mathbf{X}}\)</span>. We first decompose the covariance matrix of <span class="math inline">\({\mathbf{X}}\)</span>, <span class="math inline">\({\mathbf{\Sigma_{XX}}}\)</span>, into systematic covariance consistent with the functional group information <span class="math inline">\({\mathbf{\Sigma_{XX}}}^{(G)}\)</span> and the residual covariance not explained by the group information <span class="math inline">\({\mathbf{\Sigma_{XX}}}^{(R)}\)</span>. Following this decomposition, we estimate the cross covariance matrix by multi-response group lasso, yielding <span class="math inline">\({\mathbf{\Sigma_{XY}}}= {\mathbf{\Sigma_{XX}}}{\mathbf{B}}_{(p \times q)} = ({\mathbf{\Sigma_{XX}}}^{(G)} + {\mathbf{\Sigma_{XX}}}^{(R)}) {\mathbf{B}}_{(p \times q)}\)</span>. As a result of this decomposition, <span class="math inline">\({\mathbf{\Sigma_{XY}}}\)</span> can also be expressed as the sum of a systematic term and a residual term, breaking down the cross-covariance into a fraction attributable to pathway-level regulation and the rest. We applied the method to epigenetic regulation analysis of mRNA expression by DNA methylation in the The Cancer Genome Atlas invasive breast cancer cohort.</p>
<p><span><strong>Keywords</strong></span>. Cross covariance matrix, data integration.</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Simon, N., Friedman, J. and Hastie, T. (2013). <em>A Blockwise Descent Algorithm for Group-penalized Multiresponse and Multinomial Regression</em></p>
Koboldt,D.C. and others. (2012). <em>Nature</em>, <strong>490</strong>, 61-70.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:30 Case Room 2 (260-057)
</p>
</div>
<div id="evaluation-of-spatial-cluster-detection-method-based-on-all-geographical-linkage-patterns" class="section level2 unnumbered">
<h2>Evaluation of spatial cluster detection method based on all geographical linkage patterns</h2>
<p style="text-align:center">
Fumio Ishioka<sup>1</sup>, Jun Kawahara<sup>2</sup>, and Koji Kurihara<sup>1</sup><br />
</p>
<p><span><strong>Evaluation of spatial cluster detection method based on all geographical linkage patterns</strong></span></p>
<p>Fumio Ishioka<span class="math inline">\(^1\)</span>, Jun Kawahara<span class="math inline">\(^2\)</span> and Koji Kurihara<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Graduate School of Environmental and Life Science, Okayama University, 3-1-1, Tsushima-naka, Kita-ku, Okayama, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Graduate School of Information Science, Nara Institute of Science and Technology, Nara, Japan</p>
<p><span><strong>Abstract</strong></span>. Currently, it is becoming easier to analyze the various types of spatial data and express them visually on a map. However, it is still difficult to estimate the location of spatial clusters based on statistical evidence. The spatial scan statistic (Kulldorff 1997), which is based on the idea of maximizing the likelihood of cluster, has been widely used for spatial cluster detection method. It is important how effectively and efficiently we find a cluster whose likelihood is high, and to find such a cluster, some scan approaches are proposed. However, most of them are limited in the shape of a detected cluster, or need an unrealistic computational time if the data size is too large. The zero-suppressed binary decision diagram (ZDD) (Minato, 1993), one approach to frequent item set mining, enables us to extract all of the potential cluster areas at a realistic computational cost. In this study, we try a new way of spatial cluster detection method to detect a cluster with truly highest likelihood by applying the ZDD, and by using them, we compare and evaluate the performance of the existing scan methods.</p>
<p>1.5mm</p>
<p><span><strong>Keywords</strong></span>. Spatial cluster, Spatial scan statistic, ZDD</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Kulldorff, M. (1997). A spatial scan statistic. <em>Communications in Statistics: Theory and Methods</em>, <strong>26</strong>, 1481–1496.</p>
Minato, S. (1993). Zero-suppressed BDDs for set manipulation in combinatorial problems. <em>In: Proceedings of the 30th ACM/IEEE Design Automation Conference</em>, 272–277.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:30 Case Room 3 (260-055)
</p>
</div>
<div id="bayesian-static-parameter-inference-for-partially-observed-stochastic-systems" class="section level2 unnumbered">
<h2>Bayesian Static Parameter Inference for Partially Observed Stochastic Systems</h2>
<p style="text-align:center">
Yaxian Xu and Ajay Jasra<br /> <br />
</p>
<p><span><strong>Bayesian Static Parameter Inference for Partially Observed Stochastic Systems</strong></span></p>
<p>YAXIAN XU<span class="math inline">\(^1\)</span> and AJAY JASRA<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics &amp; Applied Probability, National University of Singapore, Singapore, 117546, SG.</p>
<p><span><strong>Abstract</strong></span>. We consider Bayesian static parameter estimation for partially observed stochastic systems with discrete-time observations. This is a very important problem, but is very computationally challenging as the associated posterior distributions are highly complex and one has to resort to discretizing the associated probability law of the underlying stochastic system and advanced Markov chain Monte Carlo (MCMC) techniques to infer the parameters. We are interested in the situation where the discretization is in multiple dimensions. For instance, for partially observed stochastic partial differential equations (SPDEs), where dicretization is in both space and time. In such cases, multi-index Monte Carlo (MIMC) is known to have the potential to reduce the computational cost for a prescribed level of error, relative to i.i.d. sampling from the most precise discretization. We demonstrate how MCMC and particularly particle MCMC can be used in the multi-index framework for Bayesian static parameter inference for the above-mentioned models. The main idea involves constructing an approximate coupling of the posterior density of the joint on the parameter and hidden space and then correcting by an importance sampling method. Our method is illustrated numerically to be preferable for inference of parameters for a partially observed SPDE.</p>
<p><span><strong>Keywords</strong></span>. Multi-index Monte Carlo, Markov chain Monte Carlo, stochastic partial differential equations</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Christophe Andrieu, Arnaud Doucet, and Roman Holenstein. (2010). Particle markov chain monte carlo methods. <em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em>, 72(3):269–342.</p>
Haji-Ali, A. L., Nobile, F. &amp; Tempone, R. (2016). Multi-Index Monte Carlo: When sparsity meets sampling. <em>Numerische Mathematik</em>, 132, 767–806.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:30 Case Room 4 (260-009)
</p>
</div>
<div id="testing-for-genetic-associations-in-arbitrarily-structured-populations-2" class="section level2 unnumbered">
<h2>Testing for genetic associations in arbitrarily structured populations</h2>
<p style="text-align:center">
Minsun Song<br /> <br />
</p>
<p><span><strong>Testing for genetic associations in arbitrarily structured populations</strong></span></p>
<p>Minsun Song<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, Sookmyung Women’s University, Seoul, 04310, Korea</p>
<p><span><strong>Abstract</strong></span>.We present a new statistical test of association between a trait and genetic markers, which we theoretically and practically prove to be robust to arbitrarily complex population structure. The statistical test involves a set of parameters that can be directly estimated from large-scale genotyping data, such as those measured in genome-wide associations studies. We also derive a new set of methodologies, called a genotype-conditional association test, shown to provide accurate association tests in populations with complex structures, manifested in both the genetic and non-genetic contributions to the trait. We demonstrate the proposed method on a large simulation study and on the real data. Our proposed framework provides a substantially different approach to the problem from existing methods.</p>
<span><strong>Keywords</strong></span>. Genome-wide association studies, Latent variable, Population structure
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:50 098 Lecture Theatre (260-098)
</p>
</div>
<div id="consistency-of-linear-mixed-effects-model-selection-with-inconsistent-covariance-parameter-estimators" class="section level2 unnumbered">
<h2>Consistency of Linear Mixed-Effects Model Selection with Inconsistent Covariance Parameter Estimators</h2>
<p style="text-align:center">
Chihhao Chang<br /> <br />
</p>
<p><span><strong>Consistency of Linear Mixed-Effects Model Selection with Inconsistent Covariance Parameter Estimators</strong></span></p>
<p>ChihHao Chang<span class="math inline">\(^1\)</span>, HsinCheng Huang<span class="math inline">\(^2\)</span> and ChingKang Ing<span class="math inline">\(^3\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Institute of Statistics, National University of Kaohsiung, Taiwan</p>
<p><span class="math inline">\(^2 \;\)</span> Institute of Statistical Science, Academia Sinica, Taiwan</p>
<p><span class="math inline">\(^3 \;\)</span> Institute of Statistics, National Tsing Hua Univeristy, Taiwan</p>
<p><span><strong>Abstract</strong></span>. For linear mixed-effects models with data collected within one cluster, the maximum likelihood estimators of covariance parameters cannot be estimated consistently. Hence the asymptotic behaviors of likelihood-based information criteria, such as Akaike’s information criterion (AIC) are rarely discussed in literature. In the contrast, the number of the clusters is generally assumed going to infinity with the sample size to guarantee the consistency of the covariance parameter estimators and thereby guarantees the consistency of the model selection procedures. In this talk, under some mild conditions, we establish asymptotic theorems for ML estimators of covariance parameters when the number of clusters is fixed. Furthermore, the asymptotic behaviors of the generalized information criterion, which includes AIC as special cases, are well studied in our research.</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Fan, Y. and Li, R. (2012). Variable selection in linear mixed effects models. In: <em>The Annals of Statistics</em>, <strong>40</strong>, 2043 - 2068.</p>
<p>Jiang, J., Rao, J. S. Gu, Z. and Nguyen, T. (2008). Fence methods for mixed model selection. In: <em>The Annals of Statistics</em>, <strong>36</strong>, 1669-1692.</p>
<p>Müller, S., Scealy, J. L. and Welsh, A. H. (2013). Model Selection in Linear Mixed Models. In: <em>Statistical Science</em>, <strong>28</strong>, 135-167.</p>
Sun, Y., Zhang, W. and Tong, H. (2007). Estimation of the covariance matrix of random effects in longitudinal studies. In: <em>The Annals of Statistics</em>, <strong>35</strong>, 2795-2814.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:50 OGGB4 (260-073)
</p>
</div>
<div id="a-study-of-the-influence-of-articles-in-the-large-scale-citation-network" class="section level2 unnumbered">
<h2>A Study of the Influence of Articles in the Large-Scale Citation Network</h2>
<p style="text-align:center">
Frederick Kin Hing Phoa<sup>1</sup> and Livia Lin Hsuan Chang<sup>2</sup><br />
</p>
<p><span><strong>A Study of the Influence of Articles in the Large-Scale Citation Network</strong></span></p>
Frederick K. H. Phoa and Livia Lin Hsuan Chang<br />
<span><em>Institute of Statistical Science, Academia Sinica, Taipei 115, Taiwan.</em></span><br />
&gt; <span><strong>Abstract:</strong></span> Nowadays there are many research metrics at &gt; the author-, article-, journal-levels, like the impact factors and &gt; many others. However, none of them possess a universally meaningful &gt; interpretation on the research influence at all levels, not mentioning &gt; that many are subject-biased and consider neighboring relations only. &gt; In this work, we introduce a new network-based research metric called &gt; the network influence. It utilizes all information in the whole &gt; network and it is universal to any levels. Due to its statistical &gt; origin, this metric is computationally efficient and statistically &gt; interpretable even if one applies it to a large-scale network. This &gt; work demonstrates the analysis of networks via network influence using &gt; a large-scale citation database called the Web of Science. By just &gt; considering the articles among statistics community in 2005-2014, the &gt; network influence of all articles are calculated and compared, &gt; resulting in a top-ten important articles that are slightly different &gt; from the list via impact factors. This metric can be easily extended &gt; to author citation network and many similar networks embedded in the &gt; Web of Science.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:50 OGGB5 (260-051)
</p>
</div>
<div id="regularized-noise-reduction-methodology-for-high-dimensional-data-1" class="section level2 unnumbered">
<h2>Regularized Noise-Reduction Methodology for High-Dimensional Data</h2>
<p style="text-align:center">
Kazuyoshi Yata and Makoto Aoshima<br /> <br />
</p>
<p><span> <strong>Regularized Noise-Reduction Methodology for High-Dimensional Data</strong> </span></p>
<p>Kazuyoshi Yata<span class="math inline">\(^1\)</span> and Makoto Aoshima<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Institute of Mathematics, University of Tsukuba, Ibaraki 305-8571, Japan</p>
<p><span><strong>Abstract</strong></span>. In this talk, we consider principal component analysis (PCA) methods in high-dimensional settings. We first consider asymptotic properties of the conventional estimator of eigenvalues. We show that the estimator is affected by the high-dimensional noise structure directly, so that it becomes inconsistent. In order to overcome such difficulties in a high-dimensional situation, Yata and Aoshima (2012) developed a new PCA method called the noise-reduction (NR) methodology. We show that the NR method can enjoy consistency properties not only for eigenvalues but also for PC directions in high-dimensional settings. The estimator of the PC directions by the NR method has a consistency property in terms of an inner product. However, it does not hold a consistency property in terms of the Euclid norm. With the help of a thresholding method, we modify the estimator and propose a regularized NR method. We show that it holds the consistency property of the Euclid norm. Finally, we check the performance of the new NR method by using microarray data sets.</p>
<p><span><strong>Keywords</strong></span>. eigenstructure, large <span class="math inline">\(p\)</span> small <span class="math inline">\(n\)</span>, PCA, spiked model</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Yata, K. and Aoshima. M. (2012). Effective PCA for high-dimension, low-sample-size data with noise reduction via geometric representations. <em>Journal of Multivariate Analysis</em>, <strong>105</strong>, 193–215.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:50 Case Room 2 (260-057)
</p>
</div>
<div id="genetic-approach-and-statistical-approach-for-association-study-on-dna-data" class="section level2 unnumbered">
<h2>Genetic Approach and Statistical Approach for Association Study on DNA Data</h2>
<p style="text-align:center">
Makoto Tomita<br /> <br />
</p>
<p><span><strong>Genetic Approach and Statistical Approach for Association Study on DNA Data</strong></span></p>
<p>Makoto Tomita <span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Clinical Research Center, Medical Hospital of Tokyo Medical and Dental University, Tokyo, 113-8519, Japan</p>
<p><span><strong>Abstract</strong></span>. Genomic information such as genome-wide association analysis (GWAS) in DNA data is very large, however if the sample size corresponding to it is not enough, as an idea to solve, the author considers by a statistical approach and a genetic approach. The former will be briefly introduced, and the latter will be mainly explained. Basically, the method of focusing genome information becomes the center of presentation.</p>
<p><span><strong>Keywords</strong></span>. genome wide association study, linkage disequilibrium, statistical power</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Tomita, M. (2013). Focusing Approach Using LD Block and Association Study with Haplotype Combination on DNA Data, In: <em>Proceedings 2013 Eleventh International Conference on ICT and Knowledge Engineering</em>, 5–10. Bangkok: IEEE Conference #32165.</p>
<p>Tomita, M. (2015). Haplotype estimation, haplotype block identification and statistical analysis for DNA data, In: <em>Conference Program and Book of Abstracts, Conference of the International Federation of Classication Societies (IFCS-2015)</em>, 227–228, Bologna.</p>
<p>Tomita, M., Hatsumichi, M. and Kurihara, K. (2008). <em>Computational Statistics and Data Analysis</em>, <strong>52</strong>(4), 1806–1820.</p>
<p>Tomita, M., Hashimoto, N. and Tanaka, Y. (2011). <em>Computational Statistics and Data Analysis</em>, <strong>55</strong>(6), 2104–2113.</p>
Tomita, M., Kubota, T. and Ishioka, F. (2015). <em>PLoS ONE</em>, <strong>10</strong>(7), e0127358.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:50 Case Room 3 (260-055)
</p>
</div>
<div id="ssrem-a-summary-statistics-based-random-effect-model-to-estimating-heritability-co-heritability-and-effect-sizes-in-gwas-data-analysis" class="section level2 unnumbered">
<h2>SSREM: A Summary-Statistics-based Random Effect Model to Estimating heritability, co-heritability and effect sizes in GWAS data analysis</h2>
<p style="text-align:center">
Jin Liu<sup>1</sup> and Can Yang<sup>2</sup><br />
</p>
<p><span><strong>SSREM: A Summary-Statistics-based Random Effect Model to Estimating heritability, co-heritability and effect sizes in GWAS data analysis</strong></span></p>
<p>Can Yang<span class="math inline">\(^1\)</span> and Jin Liu<span class="math inline">\(^{*2}\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Mathematics, Hong Kong University of Science and Technology, Hong Kong.</p>
<p><span class="math inline">\(^{*2} \;\)</span> Center for Quantitative Medicine, Duke-NUS Medical School, Singapore</p>
<p><span><strong>Abstract</strong></span>. Most existing methods for GWAS data analysis require individual-level genotype data as their input. However, it is often not easy to get access to individual-level data, due to many practical issues, such as privacy protection and disagreement on data-sharing among multiple research groups. In this talk, we introduce SSREM, a Summary-Statistics-based approach to estimating heritability, co-heritability and effect sizes in GWAS data analysis. This is achieved by Bayesian analysis with the standard random-effect prior and a summary-statistics-based likelihood function. We have implemented a parallel Gibbs sampling strategy, which allows us to handle genome-wide-scale datasets. Our analysis results suggest that summary-statistics-based analysis can achieve comparable performance to individual-level data analysis.</p>
<span><strong>Keywords</strong></span>. Summary statistics; Genome-wide association study; Probabilistic model; Gibbs sampling; Heritability; Co-heritability.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 10:50 Case Room 4 (260-009)
</p>
</div>
<div id="meta-analytic-principal-component-analysis-in-integrative-omics-application-2" class="section level2 unnumbered">
<h2>Meta-analytic Principal Component Analysis in Integrative Omics Application</h2>
<p style="text-align:center">
Sunghwan Kim<sup>1</sup> and George Tseng<sup>2</sup><br />
</p>
<p><span><strong>Meta-analytic Principal Component Analysis in Integrative Omics Application</strong></span></p>
<p>SungHwan Kim<span class="math inline">\(^1\)</span> and George C. Tseng<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, Keimyung Univerisity, Deagu 42601, South Korea</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Biostatistics, University of Pittsburgh, Pittsburgh 15213, USA</p>
<p><span><strong>Abstract</strong></span>. With the prevalent usage of microarray and massively parallel sequencing, numerous high-throughput omics datasets have become available in the public domain. Integrating abundant information among omics datasets is critical to elucidate biological mechanisms. Due to the high- dimensional nature of the data, methods such as principal component analysis (PCA) have been widely applied, aiming at effective dimension reduction and exploratory visualization. In this paper, we combine multiple omics datasets of identical or similar biological hypothesis and introduce two variations of meta-analytic framework of PCA, namely MetaPCA. Regularization is further incorporated to facilitate sparse feature selection in MetaPCA. We apply MetaPCA and sparse MetaPCA to simulations, three transcriptomic meta-analysis studies in yeast cell cycle, prostate cancer, mouse metabolism, and a TCGA pan-cancer methylation study. The result shows improved accuracy, robustness and exploratory visualization of the proposed framework.</p>
<p><span><strong>Keywords</strong></span>. principal component analysis, meta-analysis, omics data</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Flury (1984) <em>Common principal components in k groups.</em> Journal of the American Statistical Association, 79, 892–898.</p>
Krzanowski (1979) <em>Between-groups comparison of principal components.</em> Journal of the American Statistical Association, 74, 703–707
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:10 098 Lecture Theatre (260-098)
</p>
</div>
<div id="selecting-generalised-linear-models-under-inequality-constraints" class="section level2 unnumbered">
<h2>Selecting Generalised Linear Models Under Inequality Constraints</h2>
<p style="text-align:center">
Daniel Gerhard<br /> <br />
</p>
<p><span><strong>Selecting Generalised Linear Models Under Inequality Constraints</strong></span></p>
<p>Daniel Gerhard</p>
<p>School of Mathematics &amp; Statistics, University of Canterbury, Christchurch, NZ</p>
<p><span><strong>Abstract</strong></span>. Model selection by information criteria can be used to identify a single best model or for inference based on weighted support from a set of competing models, incorporating model selection uncertainty into parameter estimates and estimates of precision. Anraku (1999) proposed a modified version of the well known Akaike information criterion, selecting models in the one-way analysis of variance models when the population means are subject to monotone trends. A generalization of this order-restricted information criterion was proposed by Kuiper et al. (2011), allowing a restriction of population means by a mixture of linear equality and inequality constraints.</p>
<p>An extension to this approach is presented, applying the generalized order-restricted information criterion to model selection from a set of generalized linear models. The class of models can comprise linear equality or inequality constraints of population parameters assuming a distribution of the exponential family for the response. The methodology is illustrated using the open source environment R with the add-on package <code>goric</code>.</p>
<p><span><strong>Keywords</strong></span>. Model selection, Order-restriction, GLM</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Anraku, K. (1999). An information criterion for parameters under a simple order restriction. <em>Biometrika</em>, <strong>86</strong>, 141–152.</p>
Kuiper, R.M., Hoijtink, H., Silvapulle, M.J. (2011). An Akaike-type information criterion for model selection under inequality constraints. <em>Biometrika</em>, <strong>98</strong>, 495–501.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:10 OGGB4 (260-073)
</p>
</div>
<div id="clustering-of-research-subject-based-on-stochastic-block-model" class="section level2 unnumbered">
<h2>Clustering of research subject based on stochastic block model</h2>
<p style="text-align:center">
Hiroka Hamada<sup>1</sup>, Keisuke Honda<sup>1</sup>, Frederick Kin Hing Phoa<sup>2</sup>, and Junji Nakano<sup>1</sup><br />
</p>
<p><span><strong>Clustering of research subject based on stochastic block model</strong></span></p>
<p>Hiroka Hamada<span class="math inline">\(^1\)</span>, Keisuke Honda<span class="math inline">\(^1\)</span>, Frederick Kin Hing Phoa<span class="math inline">\(^2\)</span> and Junji Nakano <span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Institute of Statistical Mathematics, Tachikawa, Tokyo 190-8562, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Institute of Statistical Science, Academia Sinica, Nankang, Taipei 11529, Taiwan</p>
<p><span><strong>Abstract</strong></span>. In this paper, we propose a new clustering method to measure influence of papers in all areas of science. To see structure of entire relationship we apply stochastic block model (SBM) on big scale citation network data. SBM generates a matrix which divides several blocks which represent relationship among research fields. We show this matrix can be used to visual exploratory analysis. When lists of papers are mapped this matrix we can get useful information by varied locations in visually. Elastic Map is used as dimension reduction method to calculate scalar value onto onto the corresponding principal points of each papers. We demonstrate that this projection score is can be used to evaluate divergence impact of papers across all field. To illustrate one application of our method, we analyze 450k+ articles published between 1981 and 2016 Web of Science data. In this beta version of our system, Edward, probabilistic programming language is used for estimation of SBM parameters and calculation of divergence score of papers.</p>
<p><span><strong>Keywords</strong></span>. Institutional Research, Stochastic Block Model, Elastic Map</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Nowicki,K. and Snijders,T. (2001). Estimation and prediction for stochastic block structures. <em>Journal of the American Statistical Association</em>, 96, 1077–1087.</p>
<p>Gorban,A. and Zinovyev,A. (2005). Elastic Principal Graphs and Manifolds and their Practical Applications. <em>Computing</em>, 75(4), 359–379.</p>
Tran,D., Kucukelbir,A., Dieng, A.B., Rudolph,M., Liang,D. and Blei,D.M. (2016). Edward: A library for probabilistic modeling, inference, and criticism. <em>arXiv preprint arXiv:1610.09787</em>.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:10 OGGB5 (260-051)
</p>
</div>
<div id="selecting-the-number-of-principal-components-1" class="section level2 unnumbered">
<h2>Selecting the number of principal components</h2>
<p style="text-align:center">
Yunjin Choi<br /> <br />
</p>
<p><span><strong>Selecting the number of principal components</strong></span></p>
<p>Yunjin Choi<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics and Applied Probability, National University of Singapore, 129801, Singapore</p>
<p><span><strong>Abstract</strong></span>. Principal Component Analysis (PCA) is one of the most popular methods in multivariate data analysis, which can be applied to covariance matrices. Despite the popularity of the method, there is no widely adopted standard approach to select the number of principal components to retain. To address this issue, we propose a novel method utilizing the hypothesis testing framework and test whether the currently selected principal components capture all the statistically significant signals in the given data set. While existing hypothesis testing approaches do not enjoy the exact type 1 error property and lose power under some scenarios, the proposed method provides an exact type 1 error control along with decent size of power in detecting signals. Central to our work is the post-selection inference framework which facilitates valid inference after data-driven model selection; the proposed hypothesis testing method provides exact type 1 error controls by conditioning on the selection event which leads to the inference. We also introduce a possible extension of the proposed method for high-dimensional data.</p>
<span><strong>Keywords</strong></span>. Principal component analysis, post-selection inference, hypothesis testing
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:10 Case Room 2 (260-057)
</p>
</div>
<div id="symbolic-data-analytical-approach-to-unauthorized-access-logs" class="section level2 unnumbered">
<h2>Symbolic data analytical approach to unauthorized-access logs</h2>
<p style="text-align:center">
Hiroyuki Minami and Masahiro Mizuta<br /> <br />
</p>
<p><span><strong>Symbolic data analytical approach to unauthorized-access logs</strong></span></p>
<p>Hiroyuki MINAMI and Masahiro MIZUTA</p>
<p>Information Initiative Center, Hokkaido University, Sapporo 060-0811 JAPAN</p>
<p><span><strong>Abstract</strong></span>. We have been annoyed by tons of unwilling accesses in many Internet applications including SSH (Secure SHell) known as a typical remote access tool and E-mail delivery protocols. An attacked server put a report according to the configuration and the log files have grown day by day.</p>
<p>Bad accesses might be caused by computer virus and so-called <span><em>zombie</em></span>, a hi-jacked computer. We assume that the actions would have their own trends. For example, we sometimes find that a few attacks come simultaneously from only 1 site, however, we also find several attacks from a set of the sites within 1 minute or 1 day. The IP-Addresses might be variable, however, within the assigned range. It suggests that the victimizer is just one but gets an IP-Address allocation so many times.</p>
<p>To analyze the log files and give an interpretation to them, we introduce Symbolic Data Analysis (SDA) to adopt its main idea <span><em>concept</em></span>. If we configure an appropriate <span><em>concept</em></span> whose elements (<span> <em>individuals</em></span> in SDA) are IP-Address, port-numbers and attack time span, we can reveal some relationship between <span><em>concepts</em></span> and classify them into perspective. The results would give us some useful information to protect our Internet environment.</p>
<p>We discuss how we get them and the interpretation appropriately through some practical examples.</p>
<p><span><strong>Keywords</strong></span>. Invalid network access, Firewall, Massive Data Analysis</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Collins, M. (2014). <em>Network Security Through Data Analysis</em>. O’Reilly.</p>
Minami, H. and Mizuta, M. (2016). A study on the Analysis of the refused logs by Internet Firewall. <em>Proceedings of 2016 International Conference for JSCS 30th Anniversary in Seattle</em>.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:10 Case Room 3 (260-055)
</p>
</div>
<div id="forward-selection-in-regression-models-based-on-robust-estimation" class="section level2 unnumbered">
<h2>Forward selection in regression models based on robust estimation</h2>
<p style="text-align:center">
Shan Luo<sup>1</sup> and Zehua Chen<sup>2</sup><br />
</p>
<p><span><strong>Forward selection in regression models based on robust estimation</strong></span></p>
<p>Shan Luo<span class="math inline">\(^1\)</span> and Zehua Chen<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, School of Mathematical Sciences, Shanghai Jiao Tong University, Shanghai 200240, China</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Statistics and Applied Probability, National University of Singapore, Singapore 117546, Singapore</p>
<p><span><strong>Abstract</strong></span>. For the purpose of feature selection in ultra-high dimensional regression models, it is required that a sequence of candidate models and a criterion to select the “best” model from them are available. Under different scenarios, various methods have been proposed to achieve these two goals. Intuitively, it is straightforward to choose appropriate loss and penalty functions in a regularization method to accommodate specific characteristics of the data. However, the computation could be expensive for certain cases. From recent studies, we can see that sequential method is promising to produce good candidate models for ultra-high dimensional data. Moreover, it can be easily extended to complex models other than the linear regression model. In this paper, we propose a new feature selection method based on robust estimation.</p>
<span><strong>Keywords</strong></span>. Feature selection, robust estimation, sequential method.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:10 Case Room 4 (260-009)
</p>
</div>
<div id="phylogenetic-tree-based-microbiome-association-test-2" class="section level2 unnumbered">
<h2>Phylogenetic Tree-based Microbiome Association Test</h2>
<p style="text-align:center">
Sungho Won<br /> <br />
</p>
<p><span><strong>Phylogenetic Tree-based Microbiome Association Test</strong></span></p>
<p>Sungho Won<span class="math inline">\(^{1,2}\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Public Health Science, Seoul National University, Seoul, Korea</p>
<p><span class="math inline">\(^2 \;\)</span> Interdisciplinary Program for Bioinformatics, College of Natural Science, Seoul National University, Seoul</p>
<p><span><strong>Abstract</strong></span>. Microbial metagenomics data has large inter-subject variation and operational taxonomic units (OTU) for each species are usually very sparse. Because of these problems, non-parametric approaches such as Mann-Whitney U test and Wilcoxon rank-sum test have been utilized. However these approaches suffer from low statistical powers for association analyses and thus investigation on efficient statistical analyses is necessary. Main goal in my thesis is to propose phylogenetic Tree-based Microbiome Association Test (TMAT) for association analyses between microbiome abundances of each OTU and disease phenotype. Phylogenetic tree reveals similarity between different OTUs, and thus was used to provide TMAT. TMAT calculates score test statistics for each node and test statistics for all nodes are combined into a single statistics by minimum p-value or Fisher’s combing p-value method. TMAT was compared with existing methods with extensive simulations. Simulation studies show that TMAT preserves the nominal type-1 error and its statistical powers were usually much better than existing methods for considered scenarios. Furthermore it was applied to atopic diseases and found that community profiles of Enterococcus is associated.</p>
<span><strong>Keywords</strong></span>. NGS; phylogenetic treel Microbiome Association Test
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:30 098 Lecture Theatre (260-098)
</p>
</div>
<div id="statistical-generalized-derivative-applied-to-the-profile-likelihood-estimation-in-a-mixture-of-semiparametric-models" class="section level2 unnumbered">
<h2>Statistical generalized derivative applied to the profile likelihood estimation in a mixture of semiparametric models</h2>
<p style="text-align:center">
Yuichi Hirose and Ivy Liu<br /> <br />
</p>
<p><span><strong>Statistical generalized derivative applied to the profile likelihood estimation in a mixture of semiparametric models</strong></span></p>
<p>Yuichi Hirose<span class="math inline">\(^1\)</span> and Ivy Liu<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> School of Mathematics and Statistics, Victoria University of Wellington</p>
<p><span class="math inline">\(^2 \;\)</span> School of Mathematics and Statistics, Victoria University of Wellington</p>
<p><span><strong>Abstract</strong></span>. There is a difficulty in finding an estimate of variance of the profile likelihood estimator in the joint model of longitudinal and survival data. We solve the difficulty by introducing the “statistical generalized derivative”. The derivative is used to show the asymptotic normality of the estimator without assuming the second derivative of the density function in the model exists.</p>
<p><span><strong>Keywords</strong></span>. Efficiency, Efficient information bound, Efficient score, Implicitly defined function, Profile likelihood, Semi-parametric model, Joint model, EM algorithm, Mixture model</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Hsieh, F., Tseng, Y.K. and Wang, J.L. (2006). <em>Joint modeling of survival and longitudinal data: likelihood approach revisited.</em> Biometrics <span><strong>62</strong></span>, 1037–1043.</p>
<p>Hirose, Y. (2016). <em>On differentiability of implicitly defined function in semi-parametric profile likelihood estimation.</em> Bernoulli <strong>22</strong> 589–614.</p>
<span style="font-variant: small-caps;">Preedalikit et al.</span> (2016). <em>Joint modeling of survival and longitudinal ordered data using a semiparametric approach.</em> Australian &amp; New Zealand Journal of Statistics <span><strong>58</strong></span>, 153–172.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:30 OGGB4 (260-073)
</p>
</div>
<div id="structure-of-members-in-the-organization-to-induce-innovation-quantitatively-analyze-the-capability-of-the-organization" class="section level2 unnumbered">
<h2>Structure of Members in the Organization to Induce Innovation: Quantitatively Analyze the Capability of the Organization</h2>
<p style="text-align:center">
Yuji Mizukami<sup>1</sup> and Junji Nakano<sup>2</sup><br />
</p>
<p><span><strong>Structure of Members in the Organization to Induce Innovation: Quantitatively Analyze the Capability of the Organization</strong></span></p>
<p>Yuji Mizukami<span class="math inline">\(^1\)</span> and Junji Nakano<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Nihon University, 1-2-1 Izumicho, Narashino, Chiba 275-8575, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Institute of Statistical Mathematics, 10-3 Midori-cho, Tachikawa, Tokyo 190-8562, Japan</p>
<p><span><strong>Abstract</strong></span>. Innovation is the act of creating new value by using “new connection”, “new point of view”, “new way of thinking”, “new usage method” (Schumpeter 1912). In recent years, the promotion of the Innovation has been strongly encouraged. In the field of research, attempts are also being made to create new value through connection between those fields. Moreover, along with the move to promote integration among these research fields, research is being conducted to grasp and promote the degree of them. In this research, for the purpose of providing indices for measuring the degree of them, we show indices quantitatively indicating the degree of fusion in different fields and the distance between the fields. Also, we have try to present indices for grasping the whole image based on the random graph.</p>
<p><span><strong>Keywords</strong></span>. Research Metrix, Institute Research, Co-author analysis</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Wagner, C. S., Roessner, J. D., Bobb, K., Klein, J. T., Boyack, K. W., Keyton, J. and Börner, K. (2011). <em>Approaches to understanding and measuring interdisciplinary scientific research: A review of the literature, Journal of Informetrics</em>. Vol. 5, No. 1, pp. 14-26.</p>
Mizukami, Y., Mizutani, Y., Honda, K., Suzuki, S., Nakano, J. (2017). <em>An International Research Comparative Study of the Degree of Cooperation between disciplines within mathematics and mathematical sciences, Behaviormetrika</em>, <strong>1</strong>, 19 pages, On-line.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:30 OGGB5 (260-051)
</p>
</div>
<div id="towards-a-sparse-scalable-and-stably-positive-definite-inverse-covariance-estimator-1" class="section level2 unnumbered">
<h2>Towards a sparse, scalable, and stably positive definite (inverse) covariance estimator</h2>
<p style="text-align:center">
Joong-Ho Won<br /> <br />
</p>
<p><span><strong>Abstract</strong></span> High dimensional covariance estimation and graphical models is a contemporary topic in statistics and machine learning having widespread applications. The problem is notoriously difficult in high dimensions as the traditional estimate is not even positive definite. An important line of research in this regard is to shrink the extreme spectrum of the covariance matrix estimators. A separate line of research in the literature has considered sparse inverse covariance estimation which in turn gives rise to graphical models. In practice, however, a sparse covariance or inverse covariance matrix which is simultaneously well-conditioned and at the same time computationally tractable is desired. There has been little research at the confluence of these three topics. In this paper we consider imposing a condition number constraint to various types of losses used in covariance and inverse covariance matrix estimation. This extends the approach by Won, Lim, Kim, and Rajaratnam (2013) on multivariate Gaussian log likelihood. When the loss function can be decomposed as a sum of an orthogonally invariant function of the estimate and its inner product with a function of the sample covariance matrix, we show that a solution path algorithm can be derived, involving a series of ordinary differential equations. The path algorithm is at- tractive because it provides the entire family of estimates for all possible values of the condition number bound, at the same computational cost of a single estimate with a fixed upper bound. An important finding is that the proximal operator for the condition number constraint, which turns out to be very useful in regularizing loss functions that are not orthogonally invariant and may yield non-positive-definite estimates, can be efficiently computed by this path algorithm. As a concrete illustration of its practical importance, we develop an operator-splitting algorithm that imposes a guarantee of well-conditioning as well as positive definiteness to recently proposed convex pseudo-likelihood based graphical model selection methods (Zhang and Zou, 2014; Khare, Oh, and Rajaratnam, 2015).</p>
This is a joint work with Sang-Yun Oh (UC Santa Barbara) and Bala Rajaratnam (UC Davis).
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:30 Case Room 2 (260-057)
</p>
</div>
<div id="statistical-challenges-for-proteogenomic-data-analysis" class="section level2 unnumbered">
<h2>Statistical challenges for proteogenomic data analysis</h2>
<p style="text-align:center">
Yusuke Matsui<br /> <br />
</p>
<p><span><strong>Statistical challenges for proteogenomic data analysis</strong></span></p>
<p>Yusuke Matsui</p>
<p>Division of Systems Biology, Nagoya University Graduate School of Medicine, Japan</p>
<p><span><strong>Abstract</strong></span>. Large-scale integrated cancer genome and proteome characterization efforts including the cancer genome atlas and clinical proteomic tumor analysis consortium have opened unprecedented opportunities to reveal the comprehensive understandings of cancer biology. An important challenge is organizing our knowledge how the genomic events drive the proteome and phosphoproteome to form phenotypic characteristics.</p>
<p>However, connecting the genome and the proteome is not straightforward since measuring technologies for the genome and the proteome are quite different and thus the coverage that can be measured is different. Besides, the proteome data usually include massive amount of missing values. These issues include several statistical problems - massive missing data imputation, statistical models to connect the distinct datasets from different measurement technologies, and interpretable statistical models that can explain clinical outcomes.</p>
<p>Currently, we are developing proteogenomic data analysis approaches including the systematic workflow (pipeline) for the cancer protegenomic clinical data. We will present the progression of our current work as well as discussing the statistical approaches, using actual ongoing clinical dataset.</p>
<p><span><strong>Keywords</strong></span>. Bioinformatics, Proteogenomics, Cancer, Multi-omics data, Data analysis</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Philipp, M., <span><em>et al.</em></span> (2016) Proteogenomics connects somatic mutations to signalling in breast cancer. <em>Nature</em>, 534:55–62.</p>
Hui, Z., <span><em>et al.</em></span> (2016) Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer. <em>Cell</em>, 166:755–765.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:30 Case Room 3 (260-055)
</p>
</div>
<div id="feature-selection-in-high-dimensional-models-with-complex-block-structures" class="section level2 unnumbered">
<h2>Feature selection in high-dimensional models with complex block structures</h2>
<p style="text-align:center">
Zehua Chen<sup>1</sup> and Shan Luo<sup>2</sup><br />
</p>
<p><span><strong>Feature selection in high-dimensional models with complex block structures</strong></span></p>
<p>Zehua Chen<span class="math inline">\({}^1\)</span> and Shan Luo<span class="math inline">\({}^2\)</span></p>
<p><span class="math inline">\({}^1\)</span> Department of Statistics &amp; Applied Probability, NUS, 6 Science Drive 2, Singapore 117546</p>
<p><span class="math inline">\({}^2\)</span> Department of Mathematics, Shanghai Jiaotong University, Shanghai, P. R. China</p>
<p><span><strong>Abstract</strong></span>. We consider feature selection in multivariate regression models where the response variables as well as the covariates are high-dimensional and both have intrinsic group structures. The models arise naturally in many biology studies for detecting associations between multiple traits and multiple features where the traits and features are embedded in biological functioning groups such as genes or pathways. We propose a sequential procedure for selecting the feature groups based on a correlation principle. At each step of the procedure, the response groups are fitted to already selected feature groups and the residuals are obtained for the response groups, then, the feature group which has the highest correlation with the residuals of any response group is selected next. The correlation measure is the trace of the sample canonical correlation matrix between two vectors. The EBIC is used as the stopping rule of the procedure. This procedure possesses the property of selection consistency. Compared with a group penalization approach, our method is more accurate and demands much less computation.</p>
<p><span><strong>Keywords</strong></span>. Canonical correlation, correlation principle, grouped data, simultaneous feature selection, selection consistency</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Luo, S., and Chen, Z. (2017). <em>Sequential group feature selection by correlation principle in sparse high-dimensional models with complex block structures</em>. Manuscript, submitted.</p>
Li, Y., Nan, B. and Zhu, J. (2015). <em>Multivariate sparse group lasso for the multivariate multiple linear regression with an arbitrary group structure.</em> <em>Biometrics</em> <strong>71(2)</strong>, 354–363.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 11:30 Case Room 4 (260-009)
</p>
</div>
<div id="wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series-2" class="section level2 unnumbered">
<h2>Wavelet-based power transformation of non-Gaussian long memory time series</h2>
<p style="text-align:center">
Kyungduk Ko<sup>1</sup> and Chul Eung Kim<sup>2</sup><br />
</p>
<p><span><strong>Wavelet-based Power Transformation of non-Gaussian Long Memory Time Series</strong></span></p>
<p>Kyungduk Ko<span class="math inline">\(^1\)</span> and Chul Eung Kim<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Mathematics, Boise State University, Boise, ID 83725, USA</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Applied Statistics, Yeonsei University, Seoul, Korea</p>
<p><span><strong>Abstract</strong></span>. We consider a power transformation through the well-known Box-cox transformation to induce normality from non-Gaussian long memory processes and propose a Bayesian method to simultaneously estimate the transformation parameter and long memory parameter. To ease computational burdens due to the dense variance-covariance matrix of long memory time series, we base our statistical inference on the wavelet domain rather than the original data domain. For a joint estimation of the parameters of interest, posterior estimations are carried out via Markov chain Monte Carlo (MCMC). An application to German stock return data is presented.</p>
<p><span><strong>Keywords</strong></span>. Box-Cox transformation, Discrete wavelet transform, Long memory, MCMC, Normality</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Dahlhaus, R. (1990). Efficient location and regression estimation for long range dependent regression models. <em>Annuals of Statistics</em>, 23, 1029–1047.</p>
<p>Ko, K. and Lee, J. (2008). Confidence intervals for long memory regressions. <em>Statistics and Probability Letters</em>, 78, 1894–1902.</p>
Lee, J. and Ko, K. (2007). One-way analysis of variance with long memory errors and its application to stock return data. <em>Applied Stochastic Models in Business and Industry</em>, <strong>23</strong>, 493–502.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:10 OGGB4 (260-073)
</p>
</div>
<div id="dissimilarities-between-groups-of-data" class="section level2 unnumbered">
<h2>Dissimilarities between groups of data</h2>
<p style="text-align:center">
Nobuo Shimizu<sup>1</sup>, Junji Nakano<sup>1</sup>, and Yoshikazu Yamamoto<sup>2</sup><br />
</p>
<p><span><strong>Dissimilarities between groups of data</strong></span></p>
<p>Nobuo SHIMIZU<span class="math inline">\(^1\)</span>, Junji NAKANO<span class="math inline">\(^1\)</span> and Yoshikazu YAMAMOTO<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> The Institute of Statistical Mathematics, 10-3 Midori-cho, Tachikawa, Tokyo 190-8562, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Tokushima Bunri University, 1314-1 Shido, Sanuki-city, Kagawa 769-2193, Japan</p>
<p><span><strong>Abstract</strong></span>. We often have “big data” expressed by both continuous real variables and categorical variables. When their sizes are huge, it is almost impossible to see and check each individual data. Then we divide them into small number of groups which have clear domain meanings. We express each group by using information up to second order moments. For example, means, variances and covariances are used to summarize many continuous real variables, and a Burt matrix which consists of contingency tables by pairs of categorical variables are used to summarize many categorical variables. We call such a set of descriptive statistics “aggregated symbolic data (ASD)”.</p>
<p>We here propose dissimilarities between two ASDs by utilizing pseudo-likelihood ratio test statistic and chi-squared test statistic. Former one is theoretically derived and the latter one is heuristically given. We adopt two dissimilarities for clustering districts in Tokyo by ASD derived from huge real estate data.</p>
<span><strong>Keywords</strong></span>. Aggregated symbolic data, Chi-squared test statistic, clustering, pseudo-likelihood ratio test statistic
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:10 OGGB5 (260-051)
</p>
</div>
<div id="a-package-for-multiple-precision-floating-point-computation-on-r" class="section level2 unnumbered">
<h2>A package for multiple precision floating-point computation on R</h2>
<p style="text-align:center">
Ei-Ji Nakama<sup>1</sup> and Junji Nakano<sup>2</sup><br />
</p>
<p><span><strong>A package for multiple precision floating-point computation on R</strong></span></p>
<p>Ei-ji NAKAMA<span class="math inline">\(^1\)</span> and Junji NAKANO<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> COM-ONE Ltd., 2-5-1 Asahidai, Nomi-city, Ishikawa 923-1211, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> The Institute of Statistical Mathematics, 10-3 Midori-cho, Tachikawa, Tokyo 190-8562, Japan</p>
<p><span><strong>Abstract</strong></span>. As recent requirements for numerical computation performed by R become larger and more complicated, errors from floating-point arithmetic become problematic. In R, double precision floating-point arithmetic is usually performed, but it may not be adequate or precise for some situations. To avoid and detect errors of double precision floating-point arithmetic, multiple precision arithmetic is useful. Several multiple precision arithmetic packages exist on R, but their abilities are limited. Therefore we provide another multiple precision arithmetic package Rmpenv, which can handle multiple precision arithmetic for real and complex numbers, matrix product and inversion, etc. We also provide a syntactic sugar to make easy the multiple precision computation on R. We utilize a free and open source MPACK library for multiple precision arithmetic and linear algebra computation.</p>
<span><strong>Keywords</strong></span>. Double precision, floating-point arithmetic, MPACK
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:10 Case Room 3 (260-055)
</p>
</div>
<div id="cluster-wise-regression-models-combined-by-a-quasi-linear-function-1" class="section level2 unnumbered">
<h2>Cluster-wise regression models combined by a quasi-linear function</h2>
<p style="text-align:center">
Kenichi Hayashi<sup>1</sup>, Katsuhiro Omae<sup>2</sup>, and Shinto Eguchi<sup>3</sup><br />
</p>
<p><span><strong>Cluster-wise regression models combined by a quasi-linear function</strong> </span></p>
<p>Kenichi Hayashi<span class="math inline">\(^{1*}\)</span>, Katsuhiro Omae<span class="math inline">\(^{2}\)</span> and Shinto Eguchi<span class="math inline">\(^3\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Mathematics, Keio University, Yokohama, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Statistical Science, The Graduate University for Advanced Studies, Tokyo, Japan</p>
<p><span class="math inline">\(^3 \;\)</span> The Institute of Statistical Mathematics, Tokyo, Japan</p>
<p><span><strong>Abstract</strong></span>. Suppose that there are multiple heterogeneous subgroups in a dataset. In the “Big data” era, this would be a natural assumption for many fields of application such as medicine, biology, marketing, psychology, etc. Then, conventional linear regression models result in not only poor prediction performance but also misleading interpretation of analyses. In this study, we propose an extension of cluster-wise regression models <span class="math inline">\(\phi^{-1}\left(\sum_{k=1}^Kp_k(\vec{x})\phi(\vec{\beta}_k^\top\vec{x})\right)\)</span>, where <span class="math inline">\(\phi\)</span> is a strictly increasing function, <span class="math inline">\(\vec{x}\in\mathbb{R}^d\)</span>, <span class="math inline">\(\vec{\beta}_k\)</span> is a regression coefficient for <span class="math inline">\(k\)</span>th cluster and <span class="math inline">\(p_k(\vec{x})\)</span> is a non-negative function satisfying <span class="math inline">\(\sum_{k=1}^Kp_k(\vec{x})=1\)</span> for any <span class="math inline">\(\vec{x}\)</span>. We show that the proposed model has flexibility in clustering and “averaging” multiple regressors and hence includes the existing methods such as Späth (1981), DeSarbo et al. (1989) as special cases.</p>
<p><span><strong>Keywords</strong></span>. Cluster-wise regression, Generalized linear models, Population heterogeneity</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>DeSarbo, W.S., Oliver, R.L., and Rangaswamy, A. (1989). A simulated annealing methodology for clusterwise linear regression. <em>Psychometrika</em>, <strong>54</strong>, 707–736.</p>
Späth, H. (1979). Algorithm 39: Clusterwise linear regression. <em>Computing</em>, <strong>22</strong>, 367–373.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:10 Case Room 4 (260-009)
</p>
</div>
<div id="mobile-learning-in-teaching-bioinformatics-for-medical-doctors" class="section level2 unnumbered">
<h2>Mobile Learning in teaching Bioinformatics for medical doctors</h2>
<p style="text-align:center">
Taerim Lee<br /> <br />
</p>
<span><strong>Abstract</strong></span> This paper describes an implementation of mobile learning initiative in Bioinformatics Training &amp; Education Center (BITEC) for medical doctors supported from Ministry of Welfare and Public Health Korea. This project was initiated by Dept. of Bioinformatics &amp; Statistics KNOU and Dept. of Medical Informatics of SNU Medical College for training medical doctors. The high penetration rates of mobile phone subscriptions and the rapid growing of handheld users show that it is viable for making inroads towards the usage of mobile devices as an alternative learning mode for distance learning. The mobile learning initiative is aimed to encourage learning and interactions in distance learning communities aiming to bridge the transactional distances faced by the learners and adopt mobility as the key tool in Bioinformatics courses delivery. The BITEC m-Learning initiative focuses on introducing Bioinformatics using handheld devices to be made easily accessible through the use of mobile devices for ODL Bioinformatics learners who are very busy medical doctors in ubiquitous learning environment. The m-Learning approach is considered as a learning alternative to support distance learners mainly working doctors and medical researchers in Korea. This research paper discusses the implementation of the mobile e-Book approach which has better affordable, accessible and flexible educational media.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:30 OGGB4 (260-073)
</p>
</div>
<div id="interactive-visualization-of-aggregated-symbolic-data" class="section level2 unnumbered">
<h2>Interactive visualization of aggregated symbolic data</h2>
<p style="text-align:center">
Yoshikazu Yamamoto<sup>1</sup>, Junji Nakano<sup>2</sup>, and Nobuo Shimizu<sup>2</sup><br />
</p>
<p><span><strong>Interactive visualization of aggregated symbolic data</strong></span></p>
<p>Yoshikazu YAMAMOTO<span class="math inline">\(^1\)</span> , Junji NAKANO<span class="math inline">\(^2\)</span> and Nobuo SHIMIZU<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Tokushima Bunri University, 1314-1 Shido, Sanuki-city, Kagawa 769-2193, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> The Institute of Statistical Mathematics, 10-3 Midori-cho, Tachikawa, Tokyo 190-8562, Japan</p>
<p><span><strong>Abstract</strong></span>. When we have new “big data”, the first step may be to visualize them. For visualizing continuous multivariate data, interactive parallel coordinate plot is known to be appropriate. However, the number of data is huge and some variables are categorical, a simple parallel coordinate plot is not available. We propose to divide big data into rather small groups and summarize them as aggregated symbolic data (ASD), and visualize them by triangular arranged parallel coordinate plots.</p>
<p>We have developed a statistical graphics software for this purpose. Our software equips interactive operations such as selection and linked highlighting, and is written by Java, R, and big data processing technologies such as Apache Hadoop and Apache Spark.</p>
<p>Aggregated symbolic data is a set of descriptive statistics calculated by up to second order moments of variables in each group. We also propose further summarization of ASD to describe characteristics of each variable and a pair of variables for visualizing the difference among ASDs. Real example data are visualized by our software and interpreted intuitively.</p>
<span><strong>Keywords</strong></span>. Apache Hadoop, Apache Spark, Parallel coordinate plot, Symbolic data analysis.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:30 OGGB5 (260-051)
</p>
</div>
<div id="an-incomplete-data-fisher-scoring-with-an-acceleration-method" class="section level2 unnumbered">
<h2>An incomplete-data Fisher scoring with an acceleration method</h2>
<p style="text-align:center">
Keiji Takai<br /> <br />
</p>
<p><span><strong>An Incomplete-data Fisher Scoring Method with an Acceleration Method</strong> </span></p>
<p>Keiji Takai<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Faculty of Business and Commerce, Kansai University, 3-3-35 Yamate, Suita, Osaka, JAPAN</p>
<p><span><strong>Abstract</strong></span>.</p>
<p>Incomplete data complicate conventional statistical analyses because the analyses presume complete data are always available. The primary problem is the complication of the parameter estimation. The parameter estimation is based on the observed-data log-likelihood function that consists of the sum of the logarithm of the marginalized likelihood with respect to the missing values, and thus the log-likelihood function becomes complicated to handle. The EM algorithm was proposed to make it easy to handle the log-likelihood function. However, the EM algorithm still has some problems that are often criticized (McLachlan and Krishnan, 2002); namely, slow convergence and unavailability of the standard error.</p>
<p>In my talk, I propose an incomplete-data Fisher scoring (IFS) method with an acceleration method to overcome these problems. The IFS method takes a Newton-Raphson type iteration, but it produces exactly the identical sequence or an approximate sequence to the sequence produced by the EM algorithm. The notable feature of the IFS is that the IFS can accelerate itself by adjusting its steplength and can produce the standard error with the functions used only for the acceleration. The convergence rate is faster than the EM algorithm. In the talk, I provide the convergence theorem and practical examples.</p>
<p><span><strong>Keywords</strong></span>. Incomplete data, EM algorithm, Fisher scoring, acceleration method</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Barnett, J.A., Payne, R.W. and Yarrow, D. (1990). <em>Yeasts: Characteristics and identification: Second Edition.</em> Cambridge: Cambridge University Press.</p>
<p>McLachlan, G., and Krishnan, T. (2002). The EM algorithm and extensions, 2nd Edition. Wiley.</p>
<p>(ed.) Barnett, V., Payne, R. and Steiner, R. (1995). <em>Agricultural Sustainability: Economic, Environmental and Statistical Considerations</em>. Chichester: Wiley.</p>
<p>Payne, R.W. (1997). <em>Algorithm AS314 Inversion of matrices Statistics</em>, <strong>46</strong>, 295–298.</p>
Payne, R.W. and Welham, S.J. (1990). A comparison of algorithms for combination of information in generally balanced designs. In: <em>COMPSTAT90 Proceedings in Computational Statistics</em>, 297–302. Heidelberg: Physica-Verlag.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:30 Case Room 3 (260-055)
</p>
</div>
<div id="clusterwise-low-rank-correlation-analysis-based-on-majorization" class="section level2 unnumbered">
<h2>Clusterwise low-rank correlation analysis based on majorization</h2>
<p style="text-align:center">
Kensuke Tanioka<sup>1</sup>, Satoru Hiwa<sup>2</sup>, Tomoyuki Hiroyasu<sup>2</sup>, and Hiroshi Yadohisa<sup>2</sup><br />
</p>
<p><span><strong>Clusterwise low-rank correlation analysis based on majorization</strong></span></p>
<p>Kensuke Tanioka<span class="math inline">\(^{1*}\)</span>, Satoru Hiwa<span class="math inline">\(^2\)</span>,<br />
Tomoyuki Hiroyasu<span class="math inline">\(^2\)</span> and Hiroshi Yadohisa<span class="math inline">\(^3\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Clinical Study Support Center, Wakayama Medical University Hospital, Wakayama, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Biomedical Information, Doshisha University, Kyoto, Japan</p>
<p><span class="math inline">\(^3 \;\)</span> Department of Culture and Information Science, Doshisha University, , Kyoto, Japan</p>
<p><span><strong>Abstract</strong></span>. Given correlation matrices between variables of subjects and these classes of subjects, it is important to get the distinctive local networks for each class. For example, in fMRI data analysis, such the situation is observed. In concretely, each correlation matrix between regions of interests for his/her brain is observed, and each information of class is get through the experiment. In this presentation, to achieve the purpose, we proposed simultaneous analysis for both clustering of variables and low-rank approximation of correlation matrices corresponding to each class. For the estimation, we adopt the majorization algorithm based on Pietersz and Groenen (2004) and Simon and Abell (2010). Through the proposed method, we can get the distinctive sparse correlation matrices corresponding to classes, while we have to determine the number of clusters.</p>
<p><span><strong>Keywords</strong></span>. sparse estimation, clustering variables, ALS</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Pietersz, R., and Groenen, J.F (2004). <em>Rank Reduction of Correlation Matrices by Majorization.</em> Quant.Finance, <span><strong>4</strong></span>: 649–662.</p>
Simon, D., and Abell, J. (2010). <em>Majorization Algorithm for Constrained Correlation Matrix Approximation</em>, Linear Algebra and its Apprications,, <span><strong>432</strong></span>, 1152-1164.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:30 Case Room 4 (260-009)
</p>
</div>
<div id="towards-an-informal-test-for-goodness-of-fit" class="section level2 unnumbered">
<h2>Towards an informal test for goodness-of-fit</h2>
<p style="text-align:center">
Anna Fergusson and Maxine Pfannkuch<br /> <br />
</p>
<span><strong>Abstract</strong></span> Informal approaches to goodness-of-fit tests often involve examining the visual fit of the model to data ’by eye’. Such approaches are problematic for Year 13 and undergraduate students and teachers from a pedagogical perspective as key aspects such as sample size, the number of categories and expected variation of sample proportions are difficult to consider. In formal tests for goodness-of-fit a test statistic is used in reference to its sampling distribution to decide if the model distribution can be rejected. In general, a numeric test statistic does not have an obvious graphical representation within the data itself. This talk presents a new informal goodness-of-fit test that uses a simulation-based modelling tool. Drawing on ideas from graphical inference, the proposed test does not use numerical test statistics but plots as test statistics. Comparisons of performance demonstrate that the proposed test leads to similar decisions about the fit of the model distribution as the chi square goodness-of-fit test. A research study with Year 13 teachers indicated that there could be pedagogical benefits of using this informal goodness-of-fit test in terms of introducing important modelling and hypothesis test concepts.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:50 OGGB4 (260-073)
</p>
</div>
<div id="an-ewma-chart-for-monitoring-covariance-matrix-based-on-dissimilarity-index" class="section level2 unnumbered">
<h2>An EWMA chart for monitoring covariance matrix based on dissimilarity index</h2>
<p style="text-align:center">
Longcheen Huwang<br /> <br />
</p>
<p><span><strong>An EWMA Chart for Monitoring Covariance Matrix Based on Dissimilarity Index</strong></span></p>
<p>Longcheen Huwang</p>
<p>Institute of Statistics, National Tsing Hua University, Hsinchu 30043,<br />
Taiwan</p>
<p><span><strong>Abstract</strong></span>. In this talk, we propose an EWMA chart for monitoring covariance matrix based on the dissimilarity index of two matrices. It is different from the conventional EWMA charts for monitoring covariance matrix which are either based on comparing the sum or product or both of the eigenvalues of the estimated EWMA covariance matrix with those of the IC covariance matrix. The proposed chart essentially monitors covariance matrix by comparing the individual eigenvalues of the estimated EWMA covariance matrix with those of the estimated covariance matrix from the IC phase I data. We evaluate the performance of the proposed chart by comparing it with the best existing chart under the multivariate normal process. Furthermore, to prevent the control limit of the proposed EMMA chart using the limited IC phase I data from having extensively excessive false alarms, we use a bootstrap method to adjust the control limit to guarantee that the proposed chart has the actual IC average run length not less than the nominal one with a certain probability. Finally, we use an example to demonstrate the applicability and implementation of the proposed chart.</p>
<p><span><strong>Keywords</strong></span>. Average run length, dissimilarity index, EWMA; out-of-control</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Hawkins, D.M. and Maboudou-Tchao E.M. (2008). Multivariate exponentially weighted moving covariance matrix. <span><em>Technometrics</em></span>, <span><strong>50</strong></span>, 155-166.</p>
Kano, M., Hasebe, S. and Hashimoto, I. (2002). Statistical process monitoring based on dissimilarity of process data. <span><em>AIChE Journal</em></span>, <span><strong>48</strong></span>, 1231-1240.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:50 OGGB5 (260-051)
</p>
</div>
<div id="improvement-of-computation-for-nonlinear-multivariate-methods" class="section level2 unnumbered">
<h2>Improvement of Computation for Nonlinear Multivariate Methods</h2>
<p style="text-align:center">
Masahiro Kuroda<sup>1</sup>, Yuichi Mori<sup>1</sup>, and Masaya Iizuka<sup>2</sup><br />
</p>
<p><span> <strong>Improvement of Computation for Nonlinear Multivariate Methods</strong></span></p>
<p>Masahiro Kuroda<span class="math inline">\(^1\)</span>, Yuichi Mori<span class="math inline">\(^1\)</span>, and Masaya Iizuka<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Management, Okayama University of Science, Okayama, 700-0001, JAPAN</p>
<p><span class="math inline">\(^2 \;\)</span> Admission Center, Okayama University, Okayama 700-8530, JAPAN</p>
<p><span><strong>Abstract</strong></span>. Nonlinear multivariate methods (NL-MM) using optimal scaling as a quantification technique can analyze any data including quantitative and qualitative variables. The alternating least squares (ALS) algorithm is the most popular iterative algorithm in NL-MM. While the algorithm has a stable convergence property, it requires many iterations and a large computational cost, especially for a large data set involving many qualitative variables, because its convergence is linear. It is therefore important to improve the speed of computation when NL-MM with the ALS algorithm is applied. Kuroda and his co-workers tried to accelerate the convergence of the ALS algorithm using the vector <span class="math inline">\(\varepsilon\)</span> (v<span class="math inline">\(\varepsilon\)</span>) accelerator. In this talk, the v<span class="math inline">\(\varepsilon\)</span> acceleration for the ALS algorithm is implemented in NL-MM, e.g., nonlinear principal component analysis and nonlinear factor analysis, and the performances are demonstrated in numerical experiments.</p>
<p><span><strong>Keywords</strong></span>. Alternating least squares algorithm, Optimal scaling, Acceleration of convergence</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Gifi, A. (1990). <em>Nonlinear multivariate analysis</em>. Wiley.</p>
<p>Kuroda, M., Mori, Y., Iizuka, M. and Sakakihara, M. (2011). Acceleration of the alternating least squares algorithm for principal components analysis. <em>Computational Statistics and Data Analysis</em>, <strong>55</strong>, 143–153.</p>
Mori, Y., Kuroda, M. and Makino, N. (2016). <em>Nonlinear principal component analysis and its Applications</em>. JSS Research Series in Statistics, Springer.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Thursday 14<sup>th</sup> 13:50 Case Room 4 (260-009)
</p>
</div>
<div id="three-dimensional-data-visualization-education-with-virtual-reality" class="section level2 unnumbered">
<h2>Three-Dimensional Data Visualization Education with Virtual Reality</h2>
<p style="text-align:center">
Dae-Heung Jang, Jae Eun Lee, and Sojin Ahn<br /> <br />
</p>
<p><span><strong>Three-Dimensional Data Visualization Education with Virtual Reality</strong></span></p>
<p>J. E. Lee<span class="math inline">\(^1\)</span>, S. Ahn<span class="math inline">\(^1\)</span> and D. H. Jang<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, Pukyong National University, 45, Yongso-ro, Nam-gu, Busan 48513, Korea. E-mail: <a href="mailto:dhjang@pknu.ac.kr">dhjang@pknu.ac.kr</a></p>
<p><span><strong>Abstract</strong></span>. A variety of data visualization methods are utilizing to analyze huge amount of data. Among various methods, a three-dimensional image requires the rotation of the image to show stereo image on the two-dimensional screen. This study discusses data visualization education of two methods (static method and dynamic method) which make it possible to analyze the construct of stereo image to improve the restriction of the three-dimensional image display with virtual reality. This investigation can be useful to explore three-dimensional data structure more clearly.</p>
<p><span><strong>Keywords</strong></span>. Data visualization education, Virtual reality, Stereo image, R package</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Bowman, A. (2015). <em>rpanel</em>: Simple interactive controls for R using the tcltk library. R package version 1.1-3.</p>
<p>Campos, M. M. (2007). Way Cooler: PCA and Visualization Linear Algebra in the Oracle Database 2, <a href="http://oracledmt.blogspot.kr/2007/06/way-cooler-pca-and-visualization-linear.html" class="uri">http://oracledmt.blogspot.kr/2007/06/way-cooler-pca-and-visualization-linear.html</a>.</p>
<p>Ligges, U. (2017). <em>scatterplot3d</em>: 3D Scatter Plot. R package version 0.3-38.</p>
<p>Murdoch, D. (2017). <em>rgl</em>: 3D Visualization Using OpenGL. R package version 0.97.0.</p>
<p>Myers, R. H., Montgomery, D. C. and Anderson-Cook, C. M. (2016). <em>Response Surface Methodology: Process and Product Optimization Using Designed Experiments, 4th ed</em>, Wiley, New York.</p>
<p>Ripley, B. (2016). <em>MASS</em>: Support Functions and Datasets for Venables and Ripley’s MASS. R package version 7.3-47.</p>
<p>Sarkar, D. (2016). <em>lattice</em>: Trellis Graphics for R. R package version 0.20-35.</p>
<p>Soetaert, K. (2016). <em>plot3D</em>: Plotting Multi-Dimensional Data. R package version 1.1.</p>
<p>Wolf, H. P. (2015). <em>aplpack</em>: Another Plot PACKage: stem.leaf, bagplot, faces, spin3R, plotsummary, plothulls, and some slider functions. R package version 1.3.0.</p>
<p><a href="http://astrostatistics.psu.edu/datasets/SDSS" class="uri">http://astrostatistics.psu.edu/datasets/SDSS</a> quasar.html.</p>
<p><a href="http://forbes.com/mlb" class="uri">http://forbes.com/mlb</a> valuations/list.</p>
<a href="http://gartner.com/newsroom/id/3412017" class="uri">http://gartner.com/newsroom/id/3412017</a>.
<p class="pagebreak">
</p>

</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1">This work is a part of the first author’s Ph.D. thesis at Seoul National University. Research of Seongil Jo was supported by Basic Science Research Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Education (NRF-2017R1D1A3B03035235). Research of Jaeyong Lee was supported by the National Research Foundation of Korea (NRF) grant funded by the Korea government (MEST) (No. 2011-0030811).
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Tuesday 12<sup>th</sup> 10:50 Case Room 4 (260-009)
</p>
<h2 id="how-does-the-textile-set-describe-geometric-structures-of-data" class="unnumbered">How does the textile set describe geometric structures of data?</h2>
<p style="text-align:center">
Ushio Tanaka<sup>1</sup> and Tomonari Sei<sup>2</sup><br />
</p>
<p><span><strong>How does the textile set describe geometric structures of data?</strong></span><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#fnref1">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="programme-and-abstracts-for-wednesday-13th-of-december.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["NZSA-IASC-Programme.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
